\documentclass[a4paper]{report}

% Packages
\usepackage{arsclassica} 
\usepackage{graphicx}
\usepackage{epsfig}
\usepackage{pdflscape}
\usepackage{rotating}
\usepackage{listings}
\usepackage{multirow}
\usepackage{multicol}
\lstset{numbers=right, 
                numberstyle=\tiny, 
                breaklines=true,
                backgroundcolor=\color{light-gray},
                numbersep=5pt,
                xleftmargin=.25in,
                xrightmargin=.25in} 
\usepackage[english]{babel}
\usepackage{array}
\usepackage[cc]{titlepic}
\usepackage{alltt}
\usepackage{float}
\usepackage{afterpage}
\usepackage{bm}
\usepackage{amsmath}
\usepackage{mathtools}
\usepackage{tabularx}
%\usepackage{multirow}
\usepackage{fancyvrb}
\usepackage{verbatim}
%\usepackage[usenames,dvipsnames]{color}
%\usepackage{pgf}
%\usepackage{tikz}
%\usepackage{lscape}
%\usetikzlibrary{positioning,shapes.misc,backgrounds,arrows}
%\usepgflibrary{shapes.geometric}
%\usepackage{pstricks}
%\usepackage{wrapfig}
% \definecolor{lightBlue}{HTML}{B0E0E6}
% \definecolor{RoyalBlue}{HTML}{00688B}
% \definecolor{paleYellow}{HTML}{EEE9BF}
% \definecolor{light-gray}{gray}{0.95}
% \definecolor{vlight-gray}{gray}{0.97}
% \advance\textwidth 2cm
% \advance\textheight 3cm
% \advance\oddsidemargin -1.5cm
% \advance\evensidemargin -1.5cm
% \advance\topmargin -3cm
%\setlength{\parindent}{0pt}
%\setlength{\parskip}{1ex plus 0.5ex minus 0.2ex}
%\renewcommand{\baselinestretch}{1.25}
%\setcounter{tocdepth}{0}
\usepackage{setspace}

\usepackage{caption}
\usepackage{natbib}
\bibpunct{[}{]}{,}{s}{,}{,}
\usepackage{enumerate}
\usepackage{xspace}
\usepackage{titlesec}
\makeatletter
\renewcommand{\@makechapterhead}[1]{%
\vspace*{50 pt}%
{\setlength{\parindent}{0pt} \raggedright \normalfont
\bfseries\Huge
\ifnum \value{secnumdepth}>1 
   \if@mainmatter\thechapter.\ \fi%
\fi
#1\par\nobreak\vspace{40 pt}}}
\makeatother

\long\def\greybox#1{%
    \newbox\contentbox%
    \newbox\bkgdbox%
    \setbox\contentbox\hbox to \hsize{%
        \vtop{
            \kern\columnsep
            \hbox to \hsize{%
                \kern\columnsep%
                \advance\hsize by -2\columnsep%
                \setlength{\textwidth}{\hsize}%
                \vbox{
                    \parskip=\baselineskip
                    \parindent=0bp
                    #1
                }%
                \kern\columnsep%
            }%
            \kern\columnsep%
        }%
    }%
    \setbox\bkgdbox\vbox{
        \pdfliteral{0.85 0.85 0.85 rg}
        \hrule width  \wd\contentbox %
               height \ht\contentbox %
               depth  \dp\contentbox
        \pdfliteral{0 0 0 rg}
    }%
    \wd\bkgdbox=0bp%
    \vbox{\hbox to \hsize{\box\bkgdbox\box\contentbox}}%
    \vskip\baselineskip%
}



% title setup
\title{ \vspace{1in} Antecedents of higher-order chromatin structure: \\ Insights from integrative modelling}
\author{\bf Benjamin L. Moore }
\date{Supervisors: Colin A. Semple and Stuart Aitken \\ \vspace{20pt}
  \normalsize{MRC HGU, IGMM}
 \\ \today~ }
%\titlepic{\vspace{2in} \includegraphics[width=\textwidth]{figs/igmm_logo.jpg}}
%\titlepic{\vspace{3in} \includegraphics[width=.5\textwidth]{figs/MRC_logo.pdf}}

% END preamble
\begin{document}
\hyphenation{nseparable}
\doublespacing
\maketitle
% optional - table of contents
%\tableofcontents

\chapter{Methods}
\section{Input data}
\subsection{Hi-C data}
Raw Hi-C reads were downloaded from three published datasets through
GEO\cite{Barrett2013} or the SRA\cite{Leinonen2011a} with identifiers:
GSE35156 (H1 hESC), GSE18199 (K562) and SRX030113 (GM12878).  These
paired reads were mapped independently to the genome (hg19) using
{{\tt bowtie2}}\cite{Langmead2012} with default parameters. Aligned
reads were then processed into per-chromosome normalised interaction
matrices and correlation matrices using {{\tt HOMER}}.\cite{Heinz2010}
\\

For the calculation of principle component eigenvectors, correlation
matrices were built using a resolution of 1 Mb bins which pooled
interactions mapped $\pm 500$ kb, hence acting as a sliding window to
reduce boundary effects. The two principle component eigenvectors with
the highest eigenvalues were calculated, and that with the highest
correlation with PolII ChIP-seq signal (in either orientation) was
selected as the true representation of compartment
signal.\cite{Kalhor2012}

\subsection{Locus-level features}
Genome-wide ChIP-seq datasets for: 22 DNA binding proteins and 10 histone marks were made
available by the ENCODE consortium\cite{Dunham2012}
(\emph{personal communication}, Kundaje, A., Stanford University) along with DNase
I hypersensitivity and H2A.z occupancy, for each of the Tier 1 ENCODE
cell lines used in this work: H1 hESC, K562 and GM12878. These data were
pre-processed using MACS2\cite{Zhang2008} to produce fold-change relative
to input chromatin. GC content was also calculated and used in the
featureset.

\section{Modelling}
\subsection{Random Forest}\label{sec:rf}
Random Forest regression\cite{Breiman2001a} was used
as implemented in the R package \texttt{randomForest}.\cite{Liaw2002}
Parameters of $mtry = \frac{n}{3} \approx 12$ and $ntrees =
200$ were assumed as they approximate the defaults and are known to be
largely insensitive.\cite{Hastie2001}. \\

Variable importance within Random Forest regression models was
measured using mean decrease in accuracy in the out-of-bag (OOB) sample. This represents the average
difference (over the forest) between the accuracy of a tree with
permuted and unpermuted versions of a given variable, in units of mean
squared error (MSE).\cite{Cutler2007}

\subsection{Model performance}
The effectiveness of the modelling approach was measured by four
different metrics. Prediction accuracy was assessed by the Pearson correlation coefficient between the
predicted and observed eigenvectors (determined by 10-fold
cross-validation), and the root mean-squared error (RMSE) of the same
data. Classification error, when predictions where thresholded into
$A \geq 0; B < 0$, was also calculated using accuracy (\% correct
classifications or True Positives) and area under the receiver
operating characteristic (AUROC) curve (Suppl. Fig. X). Together these
give a comprehensive overview of the model performance, both in terms
of regression accuracy of the continuous eigenvector, and in how that
same model could be used to label discrete chromatin compartments. \\

For cross-application of cell type specific models, a single Random
Forest regression model was learned from all 1 Mb bins for a given
cell type. This was then used to predict all bins from each of the
other two cell types.

\section{Variable regions}
\subsection{Stratification by variability}
Median absolute deviation (MAD) was chosen as a robust measure of the
variability in a given 1 Mb block between the three primary cell types
used in this work: H1, K562 and GM12878. Blocks were ranked by this
measure and split into thirds that represented ``low'' variability
(the third of blocks with the lowest MAD), ``mid'' and ``high''
variability. Each subgroup was then independently modelled using the previously-described 
Random Forest approach (Section \ref{sec:rf}). \\

``Flipped'' regions are those whose compartment state
(\ref{sec:compartments}) differs in one cell type relative to the
other two. For example, if a 1 Mb bin was classified as ``open'' in H1
hESC and ``closed in both K562 and GM12878, this is said to be a
``flipped'' compartment (to open). 

\subsection{Enhancer enrichment}
Enhancer annotations were collected from the ChromHMM / SegWay
combined annotations in each cell type.\cite{Hoffman2013} Enhancers
were considered ``shared'' if there was an overlapping enhancer
annotation in either of the two other cell types, and labelled as
``tissue-specific'' otherwise. 

\section{Boundaries}
\subsection{TADs}
TAD boundaries were called using the software provided in Dixon
\emph{et al.}\citep{Dixon2012} using their recommended parameters. For
the generation of boundary profiles, the same parameters were used:
input features were averaged into 40 kb bins spanning $\pm500$ kb from
the boundary centre.

\subsection{Compartments}\label{sec:compartments}
Compartment boundaries were called by first training a two-state
hidden Markov model (HMM) on the compartment eigenvector and then
using the Viterbi algorithm to predict the most likely state sequence
that produced the observed values. The point at which transitions
occurred between states was taken as a boundary which was then
extended $\pm500$ kb to give a 1 Mb window in which a boundary was
though to occur. Within this window, higher resolution $40$ kb
eigenvectors were used to refine the boundary estimate to a point
between two bins where there was the lagrest observed
difference. Finally, this point was taken as the centre of a 100 kb
bin thought to contain the boundary between two compartments. For the
generation of boundary profiles for these boundaries, features were
averaged into 100 kb bins extending $\pm1.5$ Mb either side of the
boundary centre.

\section{Giemsa band comparison}
Cytogenic band data and Giemsa stain results were downloaded from the UCSC genome browser (table
{{\tt cytoBandIdeo}}). The genomic co-ordinates are an approximation of
cytogenic band data inferred from a large number of FISH
experiments.\cite{Furey2003} \\

To compare G-band boundaries with our compartment data, we allowed for
a $\pm 500$ kb inaccuracy in G-band boundary. For each G-band
boundary, the minimum absolute distance to any compartment or TAD
boundary was calculated for each cell type. To generate a null model, 


% \begin{figure}[H]
% \begin{center}
% \includegraphics[width=.65\textwidth]{../plots/pickingcomp_boundaries.pdf}
% \end{center}
% \end{figure}

%\bibliographystyle{unsrt}
\bibliographystyle{pnas2009}
\begin{small}
\bibliography{/Users/benmoore/Documents/library}
\end{small}
\end{document}
