\documentclass[a4paper,10pt,oneside]{book}

% begin preamble
\usepackage{arsclassica}    % fancy layout
\usepackage[english]{babel}\addto{\captionsenglish}{\renewcommand{\bibname}{References}}
\usepackage{caption}         % figure captions
\usepackage[square,numbers,super,sort&compress]{natbib}  % bibliography style
\usepackage[cc]{titlepic}    % enable logo on title page
\usepackage{graphicx}       % logo related

%\usepackage{}
\bibliographystyle{thesis}
% end preamble
\title{ \vspace{3.5in} Unravelling higher order genome organisation {\small [working
    title]} \\ \vspace{2em} {\large Thesis plan} }
\author{Benjamin L. Moore}
\titlepic{\vspace{2in} \includegraphics[width=\textwidth]{/Users/benmoore/hvl/1yrReport/figs/igmm.png}}

\begin{document}

\maketitle

\chapter{Introduction}

Need to introduce the seminal studies of genome organisation,
particularly those using C methods (\citet{Lieberman2011},
\citet{Dixon2012}, older 3C stuff\citep{VanSteensel2010} etc.). Detail
the fractal globule\citep{Lieberman2011} model of genome organisation,
as well as counter theories like strings, binders and switches
(SBS)\citep{Barbieri2012}. There's also growing computational literature
in calling domains, sub-domains (e.g.~Bing Ren's D.I. +
HMM\citep{Dixon2012}, but also alternative domains\citep{Filippova2014},
breakpoint algorithm etc.).

Also mention the criticisms of C
methods\cite{Gavrilov2013, Gavrilov2013a} and caveats of processing the
data (biases,\citep{Yaffe2011}
normalisation\citep{Imakaev2012, Hu2012, Hu2013} etc.).

Machine learning has been successful at calling chromatin states
(\texttt{ChromHMM}, \texttt{SegWay}, others) and generally building
models of complex biological phenomena. Explain how this type of
computational approach has led to biological insights. with particular
deference to ENCODE studies such as \citet{Dong2012}, as well as the
most recent ENCODE papers.

Potentially too far off topic but there's also a lot of literature on
the inference of 3D
conformation\cite{Varoquaux2014a, Ay2014b, Dekker2013, Bau2011a, Hu2013a}
--- that is, actual three-dimensional trajectories of the DNA polymer
--- using data generated by C based methods as input. Current results
could be extended by looking at 3D spatial data.

Finally discuss the most recent and overlapping work published, such as
that by \citet{Comoglio2014} in which replication timing was predicted
by combinatorial modelling.

\chapter{Methods}

I already have some text for this from first year report and the paper
methods section. Things to cover include:

\begin{itemize}
\itemsep1pt\parskip0pt\parsep0pt
\item
  Processing raw reads, mapping
\item
  ICE, Hi-C normalisation
\item
  Calling boundaries, HMMs
\item
  Modelling, random forests, variable importance
\item
  GLASSO, regularisation
\item
  Citations for R packages used
\item
  Package code for entire thesis?
\end{itemize}

\chapter{Results}

Here each subsection would be a separate chapter.

\section{\textbf{Early stuff:} Modelling transcription and
chromatin}\label{early-stuff-modelling-transcription-and-chromatin}

I replicated the work of \citet{Dong2012} in modelling of
transcriptional output based on a large set of ENCODE features. I
extended their work by adding new features, and dissected the ``best
bin'' approach to discover where (relative to a gene) influential
variables correlated best with expression.

We then applied the same approach to modelling a different set of data:
the A / B compartment profiles reported in \citet{Lieberman2011}. Noting
that Hi-C datasets were available for the three tier 1 ENCODE cell
lines, I applied this modelling approach to each in turn, with their own
corpus of ENCODE features. Models of this facet of high order structure
proved roughly as accurate as those of \citet{Dong2012} were for
transcriptional output.

\section{\textbf{Model dissection}: regularised models, influential
variables,
cross-application}\label{model-dissection-regularised-models-influential-variables-cross-application}

Having reasonably accurate models of chromatin organisation, it's then
of interest to understand why they are successful and if improvements
can be made. Firstly, rankings of variable importance were looked at per
cell type model. Models were also cross-applied form one cell type to
another.

We were interested in building minimal viable models, or those suitably
regularised such that accuracy was maintained while the dimensionality
of input features was minimised. To this end, we employed the Graphical
LASSO algorithm, a tuneable L1 regulariser, to reduce each model of 36
variables down to approximately 5 with little loss in predictive power.
However, this ``wrapper'' method of regularisation was independent from
the learning algorithm, hence may not represent a truly optimal subset
of features.

In order to resolve this, we employed a regularised Random Forest
algorithm\cite{deng2012, Touw2013a}, as well as a brute-force process of
constructing all possible subset models with varying numbers of
features. For example, all combinations of five variables from the
original 36 were passed to the learner and the accuracy was compared.
From this we discovered that while model performance was affected by the
number of input features, due to the pervasive multi-collinearity any
subset model of five variables would perform almost equally well. This
signalled that generated minimal viable models may provide little
additional understanding of the relationships between higher order
chromatin organisation and our locus level features.

\section{\textbf{Odds `n' ends}: TADs, boundaries, super bounds,
G-bands}\label{odds-n-ends-tads-boundaries-super-bounds-g-bands}

TADs are a well-described facet of higher order chromatin organisation
at a scale below that of nuclear compartments. We recalled these domains
in each cell type under study and compared the results. Unsurprisingly
we found TAD boundaries to be well-matched between these cell types,
confirming them as a relatively invariant level of organisation.

The boundaries of TADs have previously been reported as bound by
numerous factors, some of which (e.g.~CTCF\citep{Ong2014}) have
previously implicated roles in organising genome conformation. With a
larger set of ChIP-seq datasets available, we quantitatively tested for
enrichment or depletion of 36 DNA binding proteins and histone
modifications. This enabled us to compare enrichments across cell types
and identify those that were consistently marking these boundaries.
Further, we applied the same methodology to boundaries of compartments
and discovered similar spectra of enrichments and depletions, but at a
lower resolution --- in agreement with a ``fractal globule'' view of
genome organisation.

We investigated the idea of ``\emph{super boundaries}'' which were both
TAD and compartment boundaries. It emerged that these boundaries, though
present, did not display stark differences from non-overlapping TAD or
compartment boundaries.

We also found an agreement between A and B compartments with the
long-known Giemsa stain bands. Both were previously known to correlated
with patterns of high and low GC content (``isochores'') and more recent
work made a similar observation but did not quantitatively test the
association ({[}cite Casey's student's thesis{]}).

\section{Additional chapters from my next
project}\label{additional-chapters-from-my-next-project}

Now that a paper from my initial project is submitted, I'll be starting
a new project which should in theory fill \textasciitilde{}2 (?)
chapters. This project will continue with the genome organisation theme
and likely continue to make use of some of the reprocessed datasets I
have generated.

Potential projects include:

\begin{itemize}
\itemsep1pt\parskip0pt\parsep0pt
\item
  Investigating the contacts between (e.g.) predicted epistatic genes
\item
  Taking current cell line comparative work and looking directly at 3D
  conformation
\item
  Other ideas\ldots
\end{itemize}

\section{Collaborations and side
projects}\label{collaborations-and-side-projects}

I've also analysed related C-methods data produced by researchers in wet
lab groups:

\begin{itemize}
\item
  Adam Douglas (\textbf{Hill group}) 4C, Capture-C: Analysis of 4C
  contacts between the ZRS enhancer and the SHH gene in mouse developing
  limb bud cells. Treated experiments are awaiting sequencing. A new
  C-method, Capture-C, will also be used across this region to backup
  findings from the 4C and FISH experiments.
\item
  Iain Williamson (\textbf{Bickmore group}) 5C: Comparing contacts for
  anterior and posterior developing limb over the HoxD locus. Also
  comparing with existing mouse Hi-C data to visualise a potentially
  changing TAD structure.
\end{itemize}

\chapter{Discussion}

Summarise key results and place into broader (particularly biological)
context.

\chapter{End material}

\begin{itemize}
\itemsep1pt\parskip0pt\parsep0pt
\item
  Appendix
\item
  References
\end{itemize}


\begin{small}
\bibliography{/Users/benmoore/Documents/library,/Users/benmoore/Documents/customrefs}
\end{small}

\end{document}
