\documentclass[a4paper,10pt,oneside]{book}

% packages 
\usepackage{arsclassica}    % fancy layout
\usepackage[english]{babel}\addto{\captionsenglish}{\renewcommand{\bibname}{References}}
\usepackage{caption}         % figure captions
\usepackage[square,numbers,super,sort&compress]{natbib}  % bibliography style
\usepackage[cc]{titlepic}    % enable logo on title page
\usepackage{graphicx}       % logo related
\usepackage{standalone}

\usepackage{bm}
\usepackage{amsmath}

% thesis only
\usepackage{tocbibind}

% used in into
\usepackage{acronym}

% used in methods
\usepackage{array}
\usepackage{booktabs}
\newcolumntype{L}{>{\centering\arraybackslash}m{4cm}}
\definecolor{light-gray}{gray}{0.95}

% appendices
\usepackage{pdfpages}
\usepackage{longtable}
\usepackage[pass]{geometry}

% don't hang captions
\captionsetup{format=plain}

% bibliography
\bibliographystyle{thesis}

% title setup
\title{ \vspace{3in} Unravelling higher order chromatin organisation through statistical analysis \\ \vspace{2em} }
\author{Benjamin L. Moore}
\titlepic{\vspace{2.2in} \includegraphics[width=\textwidth]{/Users/benmoore/hvl/1yrReport/figs/igmm.png}}

\begin{document}

\frontmatter

\maketitle
\clearpage

\addcontentsline{toc}{chapter}{Declaration}
\chapter*{Declaration}

\begin{quote}
This thesis presents my own work, wherever the contributions of others were involved this is clearly indicated.
\end{quote}

\hfill ---Benjamin L. Moore (2015)

\clearpage

\addcontentsline{toc}{chapter}{Acknowledgements}
\chapter*{Acknowledgements}


\clearpage

% ---------- Abstract ---------- %
\addcontentsline{toc}{chapter}{Abstract}
\chapter*{Abstract}
Recent technological advances underpinned by high throughput sequencing have given new insights into the three-dimensional structure of mammalian genomes. Chromatin conformation assays have been the critical development in this area, particularly Hi-C, which ascertains genome-wide patterns of intra and inter-chromosomal contacts. However many open questions remain concerning the functional relevance of such higher order structure, the extent to which it varies, and how it relates to other features of the genomic and epigenomic landscape.

Current knowledge of nuclear architecture describes a hierarchical organisation ranging from small loops between individual loci, to megabase-sized self-interacting topological domains (TADs), encompassed within large multimegabase chromosome compartments. In parallel with the discovery of these strata, the ENCODE project has generated vast amounts of data through ChIP-seq, RNA-seq and other assays applied to a wide variety of cell types, forming a comprehensive bioinformatics resource.

In this work we combine Hi-C datasets describing physical genomic contacts with a large and diverse array of chromatin features derived at a much finer scale in the same mammalian cell types. These features include levels of bound transcription factors, histone modifications and expression data. These data are then integrated in a statistically rigorous way, through a predictive modelling framework from the machine learning field. These studies were extended, within a collaborative project, to encompass a dataset of matched Hi-C and expression data collected over a murine neural differentiation timecourse.

We compare higher order chromatin organisation across a variety of human cell types and find pervasive conservation of chromatin organisation at multiple scales. We also identify structurally variable regions between cell types, that are rich in active enhancers and contain loci of known cell-type specific function. We show that broad aspects of higher order chromatin organisation, such as nuclear compartment domains, can be accurately predicted in a variety of human cell types, using models based upon underlying chromatin features. We dissect these quantitative models and find them to be generalisable to novel cell types, presumably reflecting fundamental biological rules linking compartments with key activating and repressive signals. These models describe the strong interconnectedness between locus-level patterns of local histone modifications and bound factors, on the order of hundreds or thousands of basepairs, with much broader compartmentalisation of large, multi-megabase chromosomal regions.

Finally, boundary regions are investigated in terms of chromatin features and co-localisation with other known nuclear structures, such as association with the nuclear lamina. We find boundary complexity to vary between cell types and link TAD aggregations to previously described lamin-associated domains, as well as exploring the concept of super-boundaries that span multiple levels of organisation. Together these analyses lend quantitative evidence to a model of higher order genome organisation that is largely stable between cell types, but can selectively vary locally, based on the activation or repression of key loci.

\clearpage

\tableofcontents
\clearpage

% list of figures
%\addcontentsline{toc}{chapter}{\listfigurename}
\listoffigures
\clearpage

% list of tables
%\addcontentsline{toc}{chapter}{\listtablename}
\listoftables
\clearpage

% ---------- Acronyms ---------- %
\addcontentsline{toc}{chapter}{List of Acronyms}
\chapter*{List of acronyms}
% this is auto-populated from csv (above)
\begin{acronym}[chromatin] % give longest one to fix width
\acro{3C}{Chromsome conformation capture (derivatives: 4C, 5C, Hi-C)}
\acro{AUROC}{Area under the receiver operating characteristic}
\acro{CAGE}{Cap analysis of gene expression}
\acro{ChIP-seq}{Chromatin immunoprecipitation following by high-throughput sequencing}
\acro{DI}{Directionality index}
\acro{ENCODE}{The encyclopaedia of DNA elements}
\acro{ESC}{Embryonic stem cell}
\acro{FDR}{False discovery rate}
\acro{FISH}{Flurorescent \emph{in-situ} hybridisation}
\acro{GC}{Guanine and cytosine (content of a DNA sequence)}
\acro{GO}{Gene ontology}
\acro{Hi-C}{Genome-wide 3C experiment using high-throughput sequencing}
\acro{HMM}{Hidden Markov Model}
\acro{ICE}{Iterative correction and eigenvector expansion}
\acro{IF}{Interaction frequency}
\acro{MAD}{Median absolute deviation}
\acro{MSE}{Mean squared error}
\acro{OOB}{Out-of-bag}
\acro{PCC}{Pearson correlation coefficient}
\acro{PCR}{Polymerase chain reaction}
\acro{PLS}{Partial least squares}
\acro{RF}{Random Forest}
\acro{RMSE}{Root mean-squared error}
\acro{RVS}{Regions of variable structure}
\acro{TAD}{Topologically-associating domains}
\acro{TSS}{Transcription start site}
\end{acronym}
\clearpage

% ---------- Published ---------- %
\addcontentsline{toc}{chapter}{Published material}
\chapter*{Published material}

Some materials contained in this thesis have previously been published in:

\begin{quote}
Moore BL, Aitken S and Semple CA (2015) Integrative modeling reveals the principles of multi-scale chromatin boundary formation in human nuclear organization. \emph{Genome Biology}, {\bf 16}:100. \\ doi:\href{http://dx.doi.org/10.1186/s13059-015-0661-x}{10.1186/s13059-015-0661-x}
\end{quote} 

Parts of Section \ref{sec:metatads} are used in a submitted manuscript:

\begin{quote}
Fraser J, Ferrai C, Chiariello AM, Schueler M, Rito T, Laudanno G, Barbieri M, Moore BL, Aitken S,
Morris KJ, Itoh M, Kawaji H, Jaeger I, Hayashizaki Y, Carninci P, Forrest ARR, FANTOM Consortium, Semple CA, Dostie J, Pombo A, Nicodemi M (2015) Hierarchical organization
of chromosome folding and its re-organization underlies transcriptional
changes in cellular differentiation. \emph{Submitted}
\end{quote}

\clearpage

% switch to actual page numbering
\mainmatter

% ---------- Intro ---------- %
\include{0-intro/intro}

% ---------- Methods ---------- %
\input{1-methods/methods}

% ---------- Results ---------- %
\input{2-hic_comparison/hic_comparison}

\input{3-predictive_modelling/predictive_modelling.tex}

\input{4-boundaries/boundaries.tex}

% will this chapter be a goer?
%\input{5-comparative/comparative.tex}

\input{6-shh/shh.tex}

% ---------- Discussion ---------- %
\input{7-discussion/discussion.tex}

\backmatter

% Appendix with includes
\input{8-appendix/appendix.tex}

% References (grabbed from includes)
\begin{small}
%\addcontentsline{toc}{chapter}{References}
\bibliography{/Users/benmoore/Documents/library,/Users/benmoore/Documents/customrefs}
\end{small}

\end{document}
