\documentclass[a4paper,10pt,oneside]{book}

% packages 
\usepackage{arsclassica}    % fancy layout
\usepackage[english]{babel}\addto{\captionsenglish}{\renewcommand{\bibname}{References}}
\usepackage{caption}         % figure captions
\usepackage[square,numbers,super,sort&compress]{natbib}  % bibliography style
\usepackage[cc]{titlepic}    % enable logo on title page
\usepackage{graphicx}       % logo related
\usepackage{standalone}

\usepackage{bm}
\usepackage{amsmath}

% used in into
\usepackage{acronym}

% used in methods
\usepackage{array}
\usepackage{booktabs}
\newcolumntype{L}{>{\centering\arraybackslash}m{4cm}}
\definecolor{light-gray}{gray}{0.95}

% appendices
\usepackage{pdfpages}
\usepackage{longtable}
\usepackage[pass]{geometry}

% don't hang captions
\captionsetup{format=plain}

% bibliography
\bibliographystyle{thesis}

% title setup
\title{ \vspace{3in} Unravelling higher order chromatin organisation through statistical analysis \\ \vspace{2em} }
\author{Benjamin L. Moore}
\titlepic{\vspace{2.2in} \includegraphics[width=\textwidth]{/Users/benmoore/hvl/1yrReport/figs/igmm.png}}

\begin{document}

\maketitle
\clearpage

\tableofcontents
\clearpage

% list of figures
\addcontentsline{toc}{chapter}{\listfigurename}
\listoffigures
\clearpage

% list of tables
\addcontentsline{toc}{chapter}{\listtablename}
\listoftables
\clearpage

% ---------- Acronyms ---------- %
\addcontentsline{toc}{chapter}{List of Acronyms}
\chapter*{List of acronyms}
% this is auto-populated from csv (above)
\begin{acronym}[chromatin] % give longest one to fix width
\acro{3C}{Chromsome conformation capture (derivatives: 4C, 5C, Hi-C)}
\acro{AUROC}{Area under the receiver operating characteristic}
\acro{CAGE}{Cap analysis of gene expression}
\acro{ChIP-seq}{Chromatin immunoprecipitation following by high-throughput sequencing}
\acro{DI}{Directionality index}
\acro{ENCODE}{The encyclopaedia of DNA elements}
\acro{ESC}{Embryonic stem cell}
\acro{FISH}{Flurorescent \emph{in-situ} hybridisation}
\acro{GC}{Guanine and cytosine (content of a DNA sequence)}
\acro{GO}{Gene ontology}
\acro{Hi-C}{Genome-wide 3C experiment using high-throughput sequencing}
\acro{HMM}{Hidden Markov Model}
\acro{ICE}{Iterative correction and eigenvector expansion}
\acro{IF}{Interaction frequency}
\acro{MAD}{Median absolute deviation}
\acro{MSE}{Mean squared error}
\acro{OOB}{Out-of-bag}
\acro{PCC}{Pearson correlation coefficient}
\acro{PCR}{Polymerase chain reaction}
\acro{PLS}{Partial least squares}
\acro{RF}{Random Forest}
\acro{RMSE}{Root mean-squared error}
\acro{RVS}{Regions of variable structure}
\acro{TAD}{Topologically-associating domains}
\acro{TSS}{Transcription start site}
\end{acronym}
\clearpage

% ---------- Published ---------- %
\addcontentsline{toc}{chapter}{Published material}
\chapter*{Published material}

Some materials contained in this thesis have previously been published in:

\begin{quote}
Moore BL, Aitken S and Semple CA (2015) Integrative modeling reveals the principles of multi-scale chromatin boundary formation in human nuclear organization. \emph{Genome Biology}, {\bf 16}:100. \\ doi:\href{http://dx.doi.org/10.1186/s13059-015-0661-x}{10.1186/s13059-015-0661-x}
\end{quote}

\clearpage

% ---------- Intro ---------- %
\include{0-intro/intro}

% ---------- Methods ---------- %
\input{1-methods/methods}

% ---------- Results ---------- %
\input{2-hic_comparison/hic_comparison}

\input{3-predictive_modelling/predictive_modelling.tex}

\input{4-boundaries/boundaries.tex}

% will this chapter be a goer?
%\input{5-comparative/comparative.tex}

\input{6-shh/shh.tex}

% ---------- Discussion ---------- %
\input{7-discussion/discussion.tex}

% Appendix with includes
\input{8-appendix/appendix.tex}

% References (grabbed from includes)
\begin{small}
\addcontentsline{toc}{chapter}{References}
\bibliography{/Users/benmoore/Documents/library,/Users/benmoore/Documents/customrefs}
\end{small}

\end{document}
