\documentclass[a4paper,11pt,twoside,toc=listof]{scrbook}

% packages 
\usepackage{arsclassica}    % fancy layout
\usepackage[english]{babel}\addto{\captionsenglish}{\renewcommand{\bibname}{References}}
\usepackage{caption}         % figure captions
\usepackage[square,numbers,super,sort&compress]{natbib}  % bibliography style
\usepackage[cc]{titlepic}    % enable logo on title page
\usepackage{graphicx}       % logo related
\usepackage{standalone}

\usepackage{bm}
\usepackage{amsmath}

% thesis only
\usepackage[nottoc,notlof,notlot,notbib]{tocbibind}
\RequirePackage{titlesec}
\RequirePackage{soul}

% used in into
\usepackage{acronym}

% used in methods
\usepackage{array}
\usepackage{booktabs}
\newcolumntype{L}{>{\centering\arraybackslash}m{4cm}}
\definecolor{light-gray}{gray}{0.95}
\usepackage{bm}
\DeclareMathOperator*{\argmin}{arg\,min}
\usepackage{tikz-cd}
\usepackage{amssymb}

% predictive modelling 
\usepackage{multirow}

% appendices
\usepackage{pdfpages}
\usepackage{longtable}

% Margins for pretty version ::
%\usepackage[pass]{geometry}
% Margins for university regulations ::
\usepackage[top=2cm, bottom=4cm, left=4cm, right=2.5cm]{geometry}
\usepackage{setspace}
\onehalfspacing

% don't hang captions
\captionsetup{format=plain}

% bibliography
\bibliographystyle{thesis}

% title setup
\title{ \vspace{2in} Unravelling higher order chromatin organisation through statistical analysis \\ \vspace{2em} 
%\author{Benjamin L. Moore} 
Benjamin L. Moore \\
\date{} \vspace{3em}
{\normalsize Doctor of Philosophy \\ 
The University of Edinburgh \\ 2015 \\} %\vspace{5em}
}

\titlepic{\vspace{1in} \includegraphics[width=\textwidth]{/Users/benmoore/hvl/1yrReport/figs/igmm.png}}

%\usepackage{etoolbox}
% http://tex.stackexchange.com/questions/204974/how-to-change-the-appearance-of-listof-headers

%\newcommand{\hhff }{\sffamily\textls[80]{\scshape }}

% KOMA script fix
\def\bf{\normalfont\bfseries}

\begin{document}

\frontmatter

\maketitle
\clearpage

%\addcontentsline{toc}{chapter}{Declaration}
\chapter{Declaration}

\begin{quote}
This thesis presents my own work, wherever the contributions of others were involved this is clearly indicated. It has not been submitted for any other degree or professional qualification. 
\end{quote}

\hfill ---Benjamin L. Moore (2015)

\clearpage

%\addcontentsline{toc}{chapter}{Acknowledgements}
\chapter{Acknowledgements}

% Firstly, I must thank my supervisor Colin Semple for the many valuable discussions and ideas that guided this thesis. Thanks also to my secondary supervisor Stuart Aitken, an inexhaustible source of statistical expertise.

% On a personal note, it would be remiss of me not to thank my mother for her support during the past three years (not to mention the prior twenty-two). Finally, thank you Nika for the pep talks and good times along the way.

\clearpage

% ---------- Abstract ---------- %
%\addcontentsline{toc}{chapter}{Abstract}
\chapter{Abstract}
Recent technological advances underpinned by high throughput sequencing have given new insights into the three-dimensional structure of mammalian genomes. Chromatin conformation assays have been the critical development in this area, particularly Hi-C, which ascertains genome-wide patterns of intra and inter-chromosomal contacts. However many open questions remain concerning the functional relevance of such higher order structure, the extent to which it varies, and how it relates to other features of the genomic and epigenomic landscape.

Current knowledge of nuclear architecture describes a hierarchical organisation ranging from small loops between individual loci, to megabase-sized self-interacting topological domains (TADs), encompassed within large multimegabase chromosome compartments. In parallel with the discovery of these strata, the ENCODE project has generated vast amounts of data through ChIP-seq, RNA-seq and other assays applied to a wide variety of cell types, forming a comprehensive bioinformatics resource.

In this work we combine Hi-C datasets describing physical genomic contacts with a large and diverse array of chromatin features derived at a much finer scale in the same mammalian cell types. These features include levels of bound transcription factors, histone modifications and expression data. These data are then integrated in a statistically rigorous way, through a predictive modelling framework from the machine learning field. These studies were extended, within a collaborative project, to encompass a dataset of matched Hi-C and expression data collected over a murine neural differentiation timecourse.

We compare higher order chromatin organisation across a variety of human cell types and find pervasive conservation of chromatin organisation at multiple scales. We also identify structurally variable regions between cell types, that are rich in active enhancers and contain loci of known cell-type specific function. We show that broad aspects of higher order chromatin organisation, such as nuclear compartment domains, can be accurately predicted in a variety of human cell types, using models based upon underlying chromatin features. We dissect these quantitative models and find them to be generalisable to novel cell types, presumably reflecting fundamental biological rules linking compartments with key activating and repressive signals. These models describe the strong interconnectedness between locus-level patterns of local histone modifications and bound factors, on the order of hundreds or thousands of basepairs, with much broader compartmentalisation of large, multi-megabase chromosomal regions.

Finally, boundary regions are investigated in terms of chromatin features and co-localisation with other known nuclear structures, such as association with the nuclear lamina. We find boundary complexity to vary between cell types and link TAD aggregations to previously described lamin-associated domains, as well as exploring the concept of super-boundaries that span multiple levels of organisation. Together these analyses lend quantitative evidence to a model of higher order genome organisation that is largely stable between cell types, but can selectively vary locally, based on the activation or repression of key loci.

\cleardoublepage

%\addcontentsline{toc}{chapter}{Lay summary}
\chapter{Lay summary}

Each human cell contains DNA that would extend for two metres if fully straightened. Instead, this same length of DNA is highly compacted into micrometre-sized cell nuclei. Recently experimental methods such as Hi-C have been developed which allow the inspection of this folded state, generating counts of how frequently chromosomal regions are interacting with each other. These counts can be statistically analysed to reveal different levels of structures, including loops between two distant locations, knot-like domains of self-interacting regions, and broad stretches of mostly active or inactive regions. 

In this work, we bring together Hi-C datasets from several different publications and combine these with a large number of chromatin datasets that quantify, for example, levels different DNA-binding proteins as well as modifications to DNA packing histone proteins. We used these datasets to build predictive models of active and inactive states across each human chromosome in three different cell types, and achieved high predictive accuracy. We then compare and contrast these models, and use them to identify the key features which define active and inactive states. 

We also analyse the boundaries between domains and compare these across cell types. We find the domains themselves are highly conserved between cell types, but observe different chromatin features marking domain boundaries. Further collaborative work involved analysis of boundaries from Hi-C data taken over successive time points, where boundary markings were found to persist as cells differentiate from stem cells.

Overall we find the three-dimensional DNA structures within cells are highly similar even between human embryonic stem cells and cells derived from blood. Where there are differences, these areas tend to highlight biological activity specific to that cell type.

\clearpage

\chapter{Contents}

\makeatletter
\@starttoc{toc}
\makeatother
%\tableofcontents
\clearpage

\chapter{List of figures}

\makeatletter
\@starttoc{lof}% Print List of Figures
\makeatother
%\listoffigures
\clearpage

\chapter{List of tables}

\makeatletter
\@starttoc{lot}% Print List of Figures
\makeatother
%\listoftables
\clearpage

% ---------- Acronyms ---------- %
%\addcontentsline{toc}{chapter}{List of Acronyms}
\chapter{List of acronyms}

% this is auto-populated from csv (above)
\begin{acronym}[chromatin] % give longest one to fix width
\acro{3C}{Chromsome conformation capture (derivatives: 4C, 5C, Hi-C)}
\acro{AUROC}{Area under the receiver operating characteristic}
\acro{CAGE}{Cap analysis of gene expression}
\acro{ChIP-seq}{Chromatin immunoprecipitation following by high-throughput sequencing}
\acro{DI}{Directionality index}
\acro{ENCODE}{The encyclopaedia of DNA elements}
\acro{ESC}{Embryonic stem cell}
\acro{FDR}{False discovery rate}
\acro{FISH}{Flurorescent \emph{in-situ} hybridisation}
\acro{GC}{Guanine and cytosine (content of a DNA sequence)}
\acro{GO}{Gene ontology}
\acro{Hi-C}{Genome-wide 3C experiment using high-throughput sequencing}
\acro{HMM}{Hidden Markov Model}
\acro{ICE}{Iterative correction and eigenvector expansion}
\acro{IF}{Interaction frequency}
\acro{LAD}{Lamin associated domain}
\acro{MAD}{Median absolute deviation}
\acro{MSE}{Mean squared error}
\acro{NAD}{Nucleolus associated domain}
\acro{OOB}{Out-of-bag}
\acro{PCC}{Pearson correlation coefficient}
\acro{PCR}{Polymerase chain reaction}
\acro{PLS}{Partial least squares}
\acro{RF}{Random Forest}
\acro{RMSE}{Root mean-squared error}
\acro{RVS}{Regions of variable structure}
\acro{SHH}{Sonic Hedgehog}
\acro{TAD}{Topologically associating domains}
\acro{TSS}{Transcription start site}
\acro{ZRS}{Zone of polarising activity regulatory sequence}
\end{acronym}
\clearpage

% ---------- Published ---------- %
%\addcontentsline{toc}{chapter}{Published material}
\chapter{Published material}

Some materials contained in this thesis have previously been published in:

\begin{quote}
Moore BL, Aitken S and Semple CA (2015) Integrative modeling reveals the principles of multi-scale chromatin boundary formation in human nuclear organization. \emph{Genome Biology}, {\bf 16}:100. \\ doi:\href{http://dx.doi.org/10.1186/s13059-015-0661-x}{10.1186/s13059-015-0661-x}
\end{quote} 

{\flushleft Parts of Section \ref{sec:metatads} are used in a submitted manuscript:}

\begin{quote}
Fraser J, Ferrai C, Chiariello AM, Schueler M, Rito T, Laudanno G, Barbieri M, Moore BL, Aitken S,
Morris KJ, Itoh M, Kawaji H, Jaeger I, Hayashizaki Y, Carninci P, Forrest ARR, FANTOM Consortium, Semple CA, Dostie J, Pombo A, Nicodemi M (2015) Hierarchical organization
of chromosome folding and its re-organization underlies transcriptional
changes in cellular differentiation. \emph{Submitted}
\end{quote}

\clearpage

% switch to actual page numbering
\mainmatter

% ---------- Intro ---------- %
\include{0-intro/intro}

% ---------- Methods ---------- %
\input{1-methods/methods}

% ---------- Results ---------- %
\input{2-hic_comparison/hic_comparison}

\input{3-predictive_modelling/predictive_modelling.tex}

\input{4-boundaries/boundaries.tex}

% will this chapter be a goer?
%\input{5-comparative/comparative.tex}

\input{6-shh/shh.tex}

% ---------- Discussion ---------- %
\input{7-discussion/discussion.tex}

\backmatter

% Appendix with includes
\input{8-appendix/appendix.tex}

% References (grabbed from includes)
%\begin{small}
\addcontentsline{toc}{chapter}{\sffamily\textls[80]{\scshape references}}
\bibliography{/Users/benmoore/Documents/library,/Users/benmoore/Documents/customrefs}
%\end{small}

\end{document}
