\documentclass[a4paper,11pt,oneside]{book}

% packages 
\usepackage{arsclassica}    % fancy layout
\usepackage[english]{babel}\addto{\captionsenglish}{\renewcommand{\bibname}{References}}
\usepackage{caption}         % figure captions
\usepackage[square,numbers,super,sort&compress]{natbib}  % bibliography style
\usepackage[cc]{titlepic}    % enable logo on title page
\usepackage{graphicx}       % logo related

% Margins for pretty version ::
%\usepackage[pass]{geometry}
% Margins for university regulations ::
\usepackage[top=2cm, bottom=4cm, left=4cm, right=2.5cm]{geometry}
\usepackage{setspace}
\onehalfspacing

\usepackage{standalone}
\standalonetrue

% don't hang captions
\captionsetup{format=plain}

% bibliography
\bibliographystyle{../thesis}

% title setup
\title{ \vspace{3in} Unravelling higher order genome organisation {\small [working
    title]} \\ \vspace{2em} {\large {\bf Discussion}} }
\author{Benjamin L. Moore}
\titlepic{\vspace{2.2in} \includegraphics[width=\textwidth]{/Users/benmoore/hvl/1yrReport/figs/igmm.png}}

\begin{document}

%\maketitle

\chapter{Discussion}

% General, intro-y type stuff
%Since the publication of the Hi-C assay, there has been a flurry of follow-up papers largely focused on improving the technique and/or employing increasingly deep sequencing to achieve higher resolution analyses. A side-effect of this process is a growing bank of chromosome conformation datasets that, following proper reprocessing, can be compared and contrasted to gain biological insights into the importance of higher order chromatin organisation.
%
%Despite the increasing availability of 
%
%Previous studies have compared regions of higher order structure between cell types\cite{Lieberman2009} and species\cite{Dixon2012} but often anecdotally, on a selected region. Here we perform a multi-cell line comparison is a quantitative way, to 

\section{Modelling higher order chromatin organisation}

Integrative analyses of locus level chromatin data have allowed the
prediction of functional chromatin states \cite{Ernst2012, Ram2011,
  Dunham2012, Hoffman2013} but these states typically emcompass small
regions, on the order of hundreds of basepairs, rather than the larger-scale structures we are concerned with in this work.

Our data show that accurate predictions of Hi-C derived
eigenvector values, and therefore chromosome compartments, are entirely achievable. Generalisation across cell types further suggests that chromosome compartments could be inferred for those cell types without any available Hi-C data but with available ChIP-seq for a handful of chromatin features. For example, the NIH Roadmap Epigenomics project has generated histone modification data in hundreds of cell lines, tissues and developmental stages.\cite{Bernstein2010, Kundaje2015} If models in this work were adapted to use matched inputs, this would allow comprehensive comparisons of inferred chromosome compartments across a diverse range of conditions and cell types. In the same vein, chromosome compartments are known to be related to and recapitulate other aspects of higher order chromatin organisation, including replication timing domains, Lamina associated domains and nucleolus association domains. We therefore suggest a similar modelling approach could prove successful for each of these domains of interest. An exciting idea is that an integrative model capable of identifying these LADs and NADs could forward this information to a subsequent three-dimensional reconstruction algorithm, giving a truly \emph{in situ} perspective onto nuclear architecture.

We had less success with the prediction of TAD boundaries (Section \ref{sec:tadpred}). One reason for this is that the TAD calling algorithm used in this work\cite{Dixon2012} (Methods \ref{meth:tad calling}), though a published and well-used method, produces observably-flawed domain calls in some contexts. In addition the sensitivity of this method is proportional to the sample sequencing depth, which varied across our three human Hi-C datasets. We resolved TAD domains to 40 kb bins, far below the resolution of individual CTCF motifs which can generate physical domains. Indeed, given the newly-available very highly-sequenced Hi-C datasets,\cite{Rao2014} an improved method of predicting domains might start from individual ChIP-seq peaks and consider pairs of correctly-orientated CTCF motifs. In addition, any predictive model of such domains would do well to consider the hierarchical nature of chromatin organisation (exemplified by metaTADs, Section \ref{sec:metatads}) rather than seeking simple linear discretisation of chromatin fibre into adjacent domains. Finally, we note that an accurate predictive model of lower-levels of domain organisation, be they TADs or smaller physical domains, could likely recapitulate, on aggregate, broader domains such as compartments and metaTADs, culminating in a multi-scale model of nuclear architecture from the levels of kilobases up to entire chromosomes.

% Stuff from the paper ::

%The recent abundance of epigenomic data in model cell types has
%enabled accurate modelling of the transcriptional output of human
%promoters, and a rigorously quantitative assessment of the most
%influential chromatin features underlying gene expression
%\cite{Dong2012}. We have shown that it is possible to construct
%comparable models describing the features underlying higher order
%chromatin structure, and that their predictive accuracy can be
%high. Our analysis exploits Hi-C datasets that have been re-analysed,
%from the intitial sequence read mapping onwards, identically for three
%different cell types. These data were collated with 35 locus level
%ENCODE chromatin datasets, also processed identically, and matched
%across the same cell types. In common with previous studies
%\cite{Chambers2013, Dixon2012}, we observed good concordance of higher
%order chromatin structure, reflected in Hi-C data, between different
%cell types. Random forest models summarised the important
%relationships among these many variables, providing insights into the
%quantitative contributions of locus level chromatin features to higher
%order structures. Although certain features were notably more
%influential in a particular cell type, the models shared overlapping
%constellations of informative features, allowing the cross application
%of models between cell types.

%The current data suggest that the contributions of certain locus level
%chromatin features to higher order structures vary between cell
%types. Striking examples include the strong influence of H3K9me3 in
%K562 leukemia cells, and EGR1 binding in H1 hESC. EGR1 is a pivotal
%regulator of cell fate and mitogenesis with critical roles in
%development and cancer \cite{Zwang2012}. While the patterns of
%repressive H3K9me3 accumulation have been a focus in the cancer
%literature and have been proposed as a diagnostic marker in leukemia
%\cite{Muller-Tidow2010}. Similarly, the model for GM12878
%(Epstein-Barr virus transformed lymphoblasoid) cells shows a
%disproportionate influence of ATF3 binding patterns, and ATF3
%induction is a known consequence of virus transformed cells
%\cite{Hagmeyer1996}. Thus, the most cell type specific features in
%these models may be important indicators of cell type specific
%functions. These cell type specific features present a paradox, in
%view of the strong correlations in organization genome wide across
%different cell types \cite{Chambers2013, Dixon2012}, and the
%demonstration that models trained in one cell type often perform well
%with data from other cell types. These contradictory observations are
%reconciled by the presence of inter-correlated clusters of features
%underlying A and B compartments. The shifting membership of these
%clusters evidently retains enough similarity between cell types to
%enable the cross application of models.

%\section{The role of chromatin domains}

% Conservation suggests function
% How are they established, and re-established during cell cycle

\section{Domain boundaries}

% studies perturbing boundaries --- could these contain non-coding GWAS hits?
% introduce a boundary, what happens? delete a boundary what happens?

Chromatin domains have been described at multiple scales, from 5 Mb chromosome compartments\cite{Lieberman2009} down to 185 kb contact domains\cite{Rao2014} in human cells. Across all domains, questions remain about how they are constructed and maintained. Two competing ideas are that boundary elements, akin to the classic chromatin insulators, block intra-domain contacts and the spread of heterochromatin and hence create chromatin domains; however, another suggestion is that boundary regions are rather less important and in fact the unavoidable consequence of adjacent self-interacting domains, perhaps instead held together through internal enhancer--promoter interactions, among contacts.  

In favour of functional boundary elements, knockdown of CTCF has been shown to cause increased intraTAD contacts,\cite{Zuin2013} though the same study reported an orthogonal function for cohesin

The incidental boundary hypothesis is supported by data showing that deletion of specific boundary elements in insufficient to cause adjacent domains to merge,(ref XX) In addition, the majority of CTCF sites fall within TADs rather than at their boundaries (approximately $85\%$ of human CTCF sites are non-boundary\cite{Dixon2012}). Further it has been shown that the majority of enhancer--promoter contacts are tissue invariant,\cite{Bouwman2015a} hence these constitutive contacts could account for the high levels of domain conservation reported previously\cite{Lieberman2009, Dixon2012, Chambers2013, Rao2014} and in this work (Chapter \ref{chap:hiccomparison}).

As with many biological phenomena, the question of whether boundary regions or internal contacts are stabilising chromatin domains is a reductive dichotomy, and it seems likely that both boundary insulation and interTAD contacts work together to maintain chromatin domains.

\section{Domain evolution}

In this work we find an array of chromatin features that, on average, are statistically associated or excluded from TAD or compartment boundaries (Section \ref{sec:boundaryenrichments}). Among these are features with a long history of implications in chromatin organisation, including CTCF and cohesin subunit RAD21. We also report enrichments for Alu repeat elements (Section \ref{sec:repeats}) but no other repeat classes. Alu repeats and CTCF are linked by evidence that CTCF binding sites have in the past been dispersed through waves of retrosposon expansion.\cite{Schmidt2012, Nikolaev2009} Thus this suggests a model for the evolution of topological domains, whereby purifying selection removes those inserted CTCF sites which disrupt desirable regulatory environments, while those which bring-about efficient "regulon" structures are favoured. Newly-released comparative Hi-C and CTCF datasets\cite{VietriRudan2015} provide an opportunity to investigate this proposed evolutionary model. 

% Enrichment of Alu elements suggest CTCF sites inserted, bring about novel loops and domains

%Chromatin boundaries, separating TADs and nuclear compartments at
%different scales, also showed cell type specific enrichments of
%various locus level chromatin features. Across cell types, the
%complexity of boundary composition varies considerably so that only a
%few features were seen consistently enriched or depleted at
%boundaries. Peaks associated with active promoters were notable for
%both TAD and compartment boundaries in all cell types. Among the most
%influentual variables for the random forest models constructed for the
%two hematopoietic cell lines was the ubiquitous transcription factor
%YY1, which re-appeared in the analysis of chromatin boundary
%regions. Significant enrichments of YY1 were seen at TAD and nuclear
%compartment boundaries in all three cell types. Thus, the same protein
%was implicated at the level of broad genomic binding patterns (over 1
%Mb intervals) and at the level of locally enriched peaks at boundary
%regions (spanning 100-500 Kb). This is intriguing as YY1 has recently
%been shown to co-localise with the architectural protein
%CTCF \cite{Ong2014} and suggests that these proteins cooperate in the
%establishment of domain boundaries. The identification of such
%features, significantly enriched at boundary regions, provides
%potential targets for deletion in experimental studies further
%exploring the structure and function of domains
%(e.g. \cite{Nora2012}). Both cell type specific and general
%constituents of boundaries may have utility in the biomedical
%interpretation of genomic variation in noncoding regions of the
%genome.

\section{A note on causality}

% formulate problem: is genome organised by (e.g.) transcription? or does genome organisation direct transcription?

Throughout this thesis we have probed correlative relationships: those between chromatin features and expression, or higher order chromatin structure, or domain boundaries. However even the most predictive correlations make no comment on the underlying chain of causality. Whether genome organisation is a cause of consequence of the functions of underlying genetic elements remains an open question.\cite{Sexton2015}

%Is it the case that pioneer factors and chromatin remodellers make key chromatin regions accessible, allowing the association of transcription initiation complexes and other bound factors? Or instead is the genome more passively arranged, through sequestration of repeat regions with LADs and rRNA gene clusters at the nucleolus, leaving that which remains more exposed to functional binding partners? As in common in biological systems, the reality is likely that both explanations hold for different regions of the genome.

Two different approaches could be use to address the causality question. A standard rejoinder is to design wet-lab experiments, for example extending Hi-C studies to perturbation or differentiation time courses, such as that performed by collaborators in Chapter \ref{chap:boundaries}. However, another approach is first develop theoretical models which, under simulation, recapitulate observed data, and then to use these models to generate falsifiable hypotheses about the effects of specific perturbations. This latter approach is exemplified in a study by \citet{Giorgetti2014a} where the authors applied physical polymer modelling to deconvolute population-level 5C data into single-cell conformations. The model suggests that population-level averages are explained by transient contacts in each cell, rather than persistent loops. Furthermore, these models were able to predict the effects of a genetic deletion of a CTCF site and found that contacts within a TAD contribute to maintenance of the domain, dispelling an insulation-only explanation.\cite{Giorgetti2014a} This is also in agreement with experimental results showing that TADs can remain intact with the depletion of CTCF over a timecourse.\cite{Zuin2013}

The models built in this thesis could also be applied to predicting the effects of experimental perturbations. For example, an experiment decreasing the tri-methylation of H3K9, through down-regulation of SETDB1 or SV39H1 for example, might be expected to lead to heterochromatic regions to become more permissive or allow the transcription of marked tandem repeat sequences.\cite{Kim2012}  Our models further suggest the effect would be most pronounced in K562 cells (Section \ref{sec:varimp}). A previous experiment analysed the effects of losing H3K9me3 in SETDB1 knockout mice and found increased expression of a number of endogenous retroviruses,\cite{Karimi2011} but whether these expression changes were also coupled with alterations in chromosome compartment was not tested. Performing such an experiment over a number of timepoints could help to establish whether expression drives genomic regions to an active compartment or \emph{vice versa}.

%For example, in Chapter \ref{chap:boundaries} we find CTCF as strongly boundary-enriched and predictive of TAD boundaries. Indeed, this binding factor is the prime candidate for organising long-range DNA--DNA loops\cite{Rao2014, VietriRudan2015} and has been labelled the "master weaver of the genome".\cite{Phillips2009} It surprising to find that topological domains persist when CTCF is knocked-down.\cite{Zuin2013}
%
%For even greater precision, single cell Hi-C experiments with improved yield are expected to be published in the near-term, thus could allow 
%
%In Chapter \ref{chap:shh} we analyse experimental results which do attempt to address causality in a model region. Namely, is the ZRS--SHH contact itself the driver of SHH expression, or is the contact just an ever-present communication channel for the two regions.

%While many histone
%modifications can now be quantified experimentally,\cite{Nikolov2012, Sajan2012, Ernst2011} an integrated
%understanding of general mechanisms underlying the cause or effect of
%these marks lags behind. A 2011 opinion piece asked
%the question ``Histone modification: cause or
%cog?''\cite{Henikoff2011} and speculated that nucleosome modifications
%could be by-products of transcription machinery, as opposed to
%the ``histone code'' hypothesis which suggests that histone
%modifications are placed to direct alterations in chromatin
%state. This latter hypothesis is often tacitly invoked in the
%chromatin literature, wherein a mark may be described as
%``repressive'' or ``activating'' despite only the observation of a
%correlative relationship.\cite{Henikoff2011} 
%Similarly, the interplay
%between locus-level factors and higher-order organisation of
%chromatin, while known the be an important factor in
%transcription, remains poorly understood mechanisatically.\cite{Li2011} 


\section{Insights into genome organisation}

% RVS
% largely invariant but with local changes
% functional importance of higher order structure ?

Overall our results agree with a functional model of genome architecture whereby a majority of the genome is arranged into large static compartments (Section \ref{sec:wiggles}), be they Lamina associated, nucleolus associated or central and accessible chromatin. Indeed, it seems plausible that such large, constitutive anchor points may be enough to generate a significant amount of concordance in nuclear architecture between cell types.\cite{Bouwman2015a} These broad similarities are coupled with local structural changes in different cell lines (Section \ref{sec:variableregions}), allowing cell type specific regulation of gene environments through "looping out", detachment from the nuclear lamina and other conceivable mechanisms of structural variation. Whether these local changes are driven by DNA-binding proteins and chromatin remodellers or functional contacts such as enhancer--promoter interactions remains unclear, though we report enrichments for both transcriptional activity and active enhancers in variable regions, but do not observe enrichments for CTCF elements, for example (Section \ref{sec:chromstateenrich}).

\section{Conclusion}

% structure function relationships remains unclear 
It has become commonplace to discuss the multi-layered, hierarchical
organization of interphase chromosomes across strata ranging from
nuclear compartments, down to the spectra of histone modifications and
bound proteins at individual sub-genic regions. However we lack a
detailed understanding of how these strata interact. We have shown
that our perspectives of features occurring at different strata can be
bridged by modelling approaches, and the models produced can used to
explore the interrelationships between these different features
quantitatively. 

We constructed cell type specific models of nuclear
organization, as reflected in Hi-C derived eigenvector profiles, to
discover the most influential features underlying higher order
structures. We found open and closed compartments to be
well-correlated with combinatorial patterns of histone modifications
and DNA binding proteins, enabling accurate predictive models. These
models could be cross-applied successfully between cell types
highlighting constellations of common structural features associated
with different nuclear compartments as expected. Dissection of the
most influential variables also revealed important differences between
models, consistent with the known biological contrasts among these
cell types, such as the prominence of EGR1 in embryonic stem cells and
H3K9me3 in the leukaemia cell line. Investigation of regions showing
variable nuclear organization across the three cell types under study,
revealed enrichments for cell type specific enhancer activity, often
nucleated at genes with known roles in cell type specific
functions. Finally we used model predictions to examine boundary
composition between higher order domains across cell types. Among
enrichments of a large number of factors observed at different
boundaries in different cell types, CTCF and YY1 were found
consistently and may cooperate to establish domain boundaries. In
summary, we show that integrative modelling of large chromatin dataset
collections using random forests can generate useful insights into
chromosome structure and seed testable hypotheses for further
experimental studies.

\ifstandalone
\begin{small}
\bibliography{/Users/benmoore/Documents/library,/Users/benmoore/Documents/customrefs}
\end{small}
\fi

\end{document}
