<!-- Thesis plan:
Needs to be about ~10 chapters total. Write in markdown, use pandoc to output Word and LaTeX versions? -->


<!--Introduction should be ~20 pages, can be largely based on existing references from reports and the paper. -->

<p>Need to introduce the seminal studies of genome organisation, particularly those using C methods (, , older 3C stuff etc.). Detail the fractal globule model of genome organisation, as well as counter theories like strings, binders and switches (SBS). There's also growing computational literature in calling domains and various sub-domains (e.g. Bing Ren's D.I. + HMM, but also alternative domains, dynamic programming solutions etc.).</p>
<p>Also mention the criticisms of C methods and discuss the intricacies of processing the data (biases, normalisation, overcorrection etc.).</p>
<p>Machine learning has been successful at calling chromatin states (<code>ChromHMM</code>, <code>SegWay</code>, others) and generally building models of complex biological phenomena. Explain how this type of computational approach has led to biological insights. with particular deference to ENCODE studies such as , as well as the most recent ENCODE papers.</p>
<p>Potentially too far off topic but there's also a lot of literature on the inference of 3D conformation — that is, actual three-dimensional trajectories of the DNA polymer — using data generated by C based methods as input. Current results could be extended by looking at 3D spatial data.</p>
<p>Finally discuss the most recent and overlapping work published, such as that by  in which replication timing was predicted by combinatorial modelling.</p>

<p>I already have some text for this from first year report and the paper methods section. Things to cover include:</p>
<ul>
<li>Processing raw reads, mapping</li>
<li>ENCODE data processing, MACSv2 (do I mention this?)</li>
<li>ICE, Hi-C normalisation</li>
<li>Calling boundaries, HMMs</li>
<li>Modelling, random forests, variable importance</li>
<li>GLASSO, regularisation</li>
<li>Citations for R packages used</li>
<li>Package code for entire thesis?</li>
</ul>

<!-- Chapters: ~20-25 pages, 10-15 figures. Some might be shorter, 10 pages and 5 figures. -->

<p>Here each section would be a separate chapter.</p>
<h1 id="early-stuff-modelling-transcription-and-chromatin"><strong>Early stuff:</strong> Modelling transcription and chromatin</h1>
<p>I replicated the work of  in modelling of transcriptional output based on a large set of ENCODE features. I extended their work by adding new features, and dissected the &quot;best bin&quot; approach to discover where (relative to a gene) influential variables correlated best with expression.</p>
<p>We then applied the same approach to modelling a different set of data: the A / B compartment profiles reported in . Noting that Hi-C datasets were available for the three tier 1 ENCODE cell lines, I applied this modelling approach to each in turn, with their own corpus of ENCODE features. Models of this facet of high order structure proved roughly as accurate as those of  were for transcriptional output.</p>
<h1 id="model-dissection-regularised-models-influential-variables-cross-application"><strong>Model dissection</strong>: regularised models, influential variables, cross-application</h1>
<p>Having reasonably accurate models of chromatin organisation, it's then of interest to understand why they are successful and if improvements can be made. Firstly, rankings of variable importance were looked at per cell type model. Models were also cross-applied form one cell type to another.</p>
<p>We were interested in building minimal viable models, or those suitably regularised such that accuracy was maintained while the dimensionality of input features was minimised. To this end, we employed the Graphical LASSO algorithm, a tuneable L1 regulariser, to reduce each model of 36 variables down to approximately 5 with little loss in predictive power. However, this &quot;wrapper&quot; method of regularisation was independent from the learning algorithm, hence may not represent a truly optimal subset of features.</p>
<p>In order to resolve this, we employed a regularised Random Forest algorithm, as well as a brute-force process of constructing all possible subset models with varying numbers of features. For example, all combinations of five variables from the original 36 were passed to the learner and the accuracy was compared. From this we discovered that while model performance was affected by the number of input features, due to the pervasive multi-collinearity any subset model of five variables would perform almost equally well. This signalled that generated minimal viable models may provide little additional understanding of the relationships between higher order chromatin organisation and our locus level features.</p>
<h1 id="odds-n-ends-tads-boundaries-super-bounds-g-bands"><strong>Odds 'n' ends</strong>: TADs, boundaries, super bounds, G-bands</h1>
<p>TADs are a well-described facet of higher order chromatin organisation at a scale below that of nuclear compartments. We recalled these domains in each cell type under study and compared the results. Unsurprisingly we found TAD boundaries to be well-matched between these cell types, confirming them as a relatively invariant level of organisation.</p>
<p>The boundaries of TADs have previously been reported as bound by numerous factors, some of which (e.g. CTCF) have previously implicated roles in organising genome conformation. With a larger set of ChIP-seq datasets available, we quantitatively tested for enrichment or depletion of 36 DNA binding proteins and histone modifications. This enabled us to compare enrichments across cell types and identify those that were consistently marking these boundaries. Further, we applied the same methodology to boundaries of compartments and discovered similar spectra of enrichments and depletions, but at a lower resolution — in agreement with a &quot;fractal globule&quot; view of genome organisation.</p>
<p>We investigated the idea of &quot;<em>super boundaries</em>&quot; which were both TAD and compartment boundaries. It emerged that these boundaries, though present, did not display stark differences from non-overlapping TAD or compartment boundaries.</p>
<p>We also found an agreement between A and B compartments with the long-known Giemsa stain bands. Both were previously known to correlated with patterns of high and low GC content (&quot;isochores&quot;) and more recent work made a similar observation but did not quantitatively test the association ([cite Casey's student's thesis]). This also hints at a linke between interphase genome organisation (i.e. that measured on average by Hi-C) and metaphase organisation, as assayed by Giemsa stain. This potentially contrasts with prominent research which found total structural rearrangement to a homogenous state during mitosis .</p>
<h1 id="additional-chapters-from-my-next-project">Additional chapters from my next project</h1>
<p>Now that a paper from my initial project is submitted, I'll be starting a new project which should in theory fill ~2 (?) chapters. This project will continue with the genome organisation theme and likely continue to make use of some of the reprocessed datasets I have generated.</p>
<p>Potential projects include:</p>
<ul>
<li>Investigating the contacts between (e.g.) predicted epistatic genes</li>
<li>Taking current cell line comparative work and looking directly at 3D conformation</li>
<li>Other ideas</li>
</ul>
<h1 id="collaborations-and-side-projects">Collaborations and side projects</h1>
<p>I've also analysed related C-methods data produced by researchers in wet lab groups:</p>
<ul>
<li><p>Adam Douglas (<strong>Hill group</strong>) 4C, Capture-C: Analysis of 4C contacts between the ZRS enhancer and the SHH gene in mouse developing limb bud cells. Treated experiments are awaiting sequencing. A new C-method, Capture-C, will also be used across this region to backup findings from the 4C and FISH experiments.</p></li>
<li><p>Iain Williamson (<strong>Bickmore group</strong>) 5C: Comparing contacts for anterior and posterior developing limb over the HoxD locus. Also comparing with existing mouse Hi-C data to visualise a potentially changing TAD structure.</p></li>
</ul>

<p>Summarise key results and place into broader (particularly biological) context.</p>
<p>Some points from the submitted manuscript:</p>
<ul>
<li>We have shown that it is possible to construct comparable models describing the features underlying higher order chromatin structure, and that their predictive accuracy can be high.</li>
<li>Random Forest models summarised the important relationships among these many variables, providing insights into the quantitative contributions of locus level chromatin features to higher order structures. Although certain features were notably more influential in a particular cell type, the models shared overlapping constellations of informative features, allowing the cross application of models between cell types.</li>
<li>These models also allowed us to probe the features underlying regions with variable higher order structure between cell types, revealing enrichments of cell type specific enhancer activity, and suggesting links between functional chromatin states and higher order domain dynamics.</li>
<li>Chromatin boundaries, separating TADs and nuclear compartments at different scales, also showed cell type specific enrichments of various locus level chromatin features. Across cell types, the complexity of boundary composition varies considerably so that only a few features were seen consistently enriched or depleted at boundaries.</li>
</ul>

<ul>
<li>Appendix</li>
<li>References</li>
</ul>
