\documentclass[a4paper,11pt,oneside]{book}

% packages 
\usepackage{arsclassica}    % fancy layout
\usepackage[english]{babel}\addto{\captionsenglish}{\renewcommand{\bibname}{References}}
\usepackage{caption}         % figure captions
\usepackage[square,numbers,super,sort&compress]{natbib}  % bibliography style
\usepackage[cc]{titlepic}    % enable logo on title page
\usepackage{graphicx}       % logo related

\usepackage{standalone}
\standalonetrue

% Margins for pretty version ::
%\usepackage[pass]{geometry}
% Margins for university regulations ::
\usepackage[top=2cm, bottom=4cm, left=4cm, right=2.5cm]{geometry}
\usepackage{setspace}
\onehalfspacing

% don't hang captions
\captionsetup{format=plain}

% bibliography
\bibliographystyle{../thesis}

% title setup
\title{ \vspace{3in} Unravelling higher order genome organisation {\small [working
    title]} \\ \vspace{2em} {\large {\bf Results 5: Collaborations}} }
\author{Benjamin L. Moore}
\titlepic{\vspace{2.2in} \includegraphics[width=\textwidth]{/Users/benmoore/hvl/1yrReport/figs/igmm.png}}

\begin{document}

%\maketitle

\chapter{Local chromatin conformation}\label{chap:shh}

\section{Introduction}

The Hi-C assay provides a genome-wide overview of chromatin conformation, however this broad scope imposes resolution limits inherent to an all-vs-all assay. For a closer look at chromatin conformation within a region of interest, alternative C-based assays such as 3C, 4C and 5C can be employed alongside classical microscopy techniques like FISH.

Here I discuss two collaborative projects involving the use of 4C and 5C data to "zoom in" on two well-studied regions related to limb development: the ZRS enhancer and HoxD gene cluster.

\section{4C at the SHH locus}

\begin{figure}
\begin{center} 
\includegraphics[width=3.5in]{figs/shhtad.pdf}
\captionsetup{width=\textwidth} 
\caption[SHH--ZRS contacts occur within a stable TAD.]{ {\bf SHH--ZRS contacts occur within a stable TAD. }
An approximately 1 Mb region of the mouse genome is shown below a Hi-C contact map (derived from previously published data\cite{Dixon2012}). A clear TAD can be identified spanning from SHH to ZRS, dashed lines show TAD boundaries called by \citet{Dixon2012}. This figure was generated for \citet{Anderson2014a}.
}\label{fig:shhtad}
\end{center} 
\end{figure} 

% , and is conserved across mammals and fish

Anterior-posterior patterning in the developing limb is regulated in mammals by \emph{Sonic hedgehog} (SHH).\cite{Anderson2012} Specifically, the SHH gene is expressed within a confined region named the "zone of polarising activity". Its expression within this region is known to be regulated by a well-studied enhancer, the "zone of polarising activity regulatory sequence" or ZRS.\cite{Hill2013a} ZRS is located almost 1 Mb downstream of its target SHH promoter in humans, and is located in intronic regions of another gene, LMBR1 (Fig. \ref{fig:shhtad}).\cite{Hill2013a, Laurell2012} Single point mutations and short insertions within this enhancer have been linked to various limb deformities, including pre- and post-axial polydactyly.\cite{Anderson2012, Lettice2008, Laurell2012} For example, a heritable point mutation in the ZRS enhancer is the cause of polydactyly in "Hemingway cats", a large group of domestic cats with extra toes that reside at the former home of Ernest Hemingway.\cite{Lettice2008}  

Collaborators have developed a model system which allows inducible SHH expression in a non-expressing 14fp cell line derived from the developing limb bud. Application of trichostatin A (TSA) then leads to detectable SHH expression, and increased levels of the histone activation mark H3K27ac at the ZRS (\emph{unpublished data}). However, the question remains whether this TSA treatment is fundamentally altering local chromatin structure, that is, bringing together the ZRS enhancer with its target SHH promoter, or whether ZRS and SHH are in contact in both the active and non-expressing cell lines and SHH expression is blocked through other means. Analysis of the region through FISH implies similar levels of compaction in SHH expressing and non-expressing cells (\emph{data not shown}), suggesting the latter explanation. Additionally, studies have previously reported the SHH--ZRS interaction is an example of a "pre-formed loop", and as such is maintained regardless of transcriptional activity.\cite{Bouwman2015a}

My part in this collaboration was to analyse 3C-seq (also known as 4C) data generated by our collaborators for the SHH--ZRS region in mouse. Additionally, the 4C procedure\cite{Stadhouders2013} was adapted for specific in-house sequencing instruments (an Ion Torrent Ion Proton\textsuperscript{TM} sequencer as opposed to Illumina\textsuperscript{TM} technology) and as such required diagnostics to confirm the experimental data was accurate. 

\subsection{4C pipeline}

The 4C analysis pipeline, starting from de-multiplexed \texttt{fastq} files as produced by our in-house sequencing facilities, can be summarised as:

\begin{enumerate}
\item Trim known bait sequence using \texttt{cutadapt},\cite{cutadapt} select only those reads where known sequence was present
\item Map reads to reference genome \texttt{mm9} using \texttt{bowtie2}\cite{Langmead2012} with the \texttt{very-sensitive} flag
\item Filter alignments with a MAPQ score $<30$ to select for high-confidence alignments using \texttt{samtools}\cite{Li2009}
\item Normalise and analyse contacts using the \texttt{r3cseq} R package (Methods \ref{methods:4c})
\end{enumerate}

% Adam sent a powerpoint he gave for section meeting, see also Prof Hill's publications

\subsection{Analysis of ZRS interactions}

\begin{figure}
\begin{center} 
\includegraphics[width=5in]{figs/shharc_full.pdf}
\captionsetup{width=\textwidth} 
\caption[TSA treatment induces a strong ZRS--SHH interaction. ]{ {\bf TSA treatment induces a strong ZRS--SHH interaction. }
4C interactions are shown as edges from source node (ZRS enhancer bait fragment) to targets along an approximately 2 Mb region of chromosome 5. Edge width is proportional to the number of interactions, only highly significant interactions are shown (FDR $q$-value $<5 \times 10 ^{-5}$). Zoomed region shows the number of interactions of the bait region with SHH in both untreated and TSA treated (after 18h) samples. Each rectangle is a restriction fragment, coloured by FDR $q$-value indicating the significance of the interaction above expected levels.
}\label{fig:ssharc}
\end{center} 
\end{figure} 

4C experiments were performed by collaborators using the ZRS region as a bait sequence, or "viewpoint", such that it contacts were measured with all other HindIII restriction fragments genome-wide. 4C was performed in both untreated and non-SHH expressing cells (\emph{TSA--}) and in cells treated with TSA, thereby causing SHH expression (\emph{TSA+}). 

The first stage in analysing these contacts is to convert observed raw sequencing reads to normalised frequencies (Methods \ref{methods:4cnorm}), these normalised values are then assigned significance scores in the form of $q$-values, with the aim of finding those significantly over-represented relative to expectation (Methods \ref{methods:4csignif}).

The results of a comparison between TSA treated and untreated samples is shown in Figure \ref{fig:ssharc}. In it we see a striking and highly significant ZRS--SHH contact in the treated sample ($q$-value $ < 5 \times 10^{-10}$), with a weaker but still significant contact in the adjacent restriction fragment in the untreated sample ($q$-value $ < 5 \times 10^{-5}$). 

We also see more broadly a much higher total number of significant contacts in the untreated sample around the viewpoint (Fig. \ref{fig:4carcs}). In the treated samples, only a few contacts cross the stringent $q$-value threshold, with the SHH--ZRS contact among the most significant in terms of both $q$-value and supporting number of reads (indicated by the thickness of the arc, Figs. \ref{fig:ssharc}, \ref{fig:4carcs}).

Existing multi-probe FISH data produced by our collaborators shows approximately equal levels of compaction in this region in both TSA treated and untreated cells (\emph{data not shown}). This information in combination with the 4C results reported here (Fig. \ref{fig:ssharc}) support a hypothesis that while both samples are held together in a TAD (Fig. \ref{fig:shhtad}) --- unavoidably inducing many contacts --- it is only in the treated sample where a highly-specific ZRS--SHH contact occurs and potentially is then bring about expression of the \emph{SHH} gene.

% Conclusions: appears ZRS contacts become more targetted in TSA+ cells, before more diffuse. 
% Maybe in contact all the time (supported by FISH data) but not engaging in specific contacts when non-expressing.

\begin{figure}
\begin{center} 
\includegraphics[width=5in]{figs/4c_arcs.pdf}
\captionsetup{width=\textwidth} 
\caption[ A stable ZRS--SHH interaction is coupled with reduced extraneous contacts. ]{ {\bf A stable ZRS--SHH interaction is coupled with reduced extraneous contacts. }
Arc plots are shown for an untreated, non-expressing 14fp cell population (\emph{untreated}) and following TSA treatment after 18 and 24 hours. Arcs link highly significant interactions ($q$-value $< 5 \times 10^{-5}$) and arc widths are proportional to the normalised number of reads recorded for the interaction (Methods \ref{methods:4c}).
}\label{fig:4carcs}
\end{center} 
\end{figure} 

%\subsection{4C / Hi-C comparison}
%
%Hi-C data in mouse cells has been previously published,\cite{Dixon2012} so can be compared with this novel 4C data to give broader contextual information about chromatin conformation in the region under study.

\subsection{Assay diagnostics}

The 4C protocol used by our collaborators in this work was that of \citet{Stadhouders2013}. In it, the authors advise some statistical tests to ensure the quality of the experiment results. Among these were:\cite{Stadhouders2013}

\begin{enumerate}
\item Sequencing reads should be found to have high duplication rates of $95\%$ or greater.
\item $50\%$ or more of all reads should map to the chromosome on which the bait region is located.
\end{enumerate}

% duplication: treated 02 : 71.2 % (reads 42.2M)
%                                 04 : 84.4 % (reads 24.2M)
%                    haeiii: 74.0% (reads 10.4M)
%                    mluci: 62.8% (reads 10M)

Sequence duplication levels were measured with \texttt{FastQC}\cite{fastqc} and are shown in Table \ref{tab:4c}. We find slightly lower than expected levels of duplication, ranging from $62.8\%$ to $84.4\%$. This suggests that while the assay does appear to be working, there may be extraneous noise and non-bait interactions in the sequencing library.

\begin{table}[]
\centering
\caption[4C sequencing library statistics.]{ {\bf 4C sequencing library statistics.}
4C experiments are summarised as total number of reads in each experiment and the percentage of those reads labelled "duplicates". Note in 4C these duplicates are not artifactual and instead result from large numbers of contacts nearby to the viewpoint.
}
\label{tab:4c}
\begin{tabular}{lcccc}
                                      & \multicolumn{2}{c}{{\bf TSA-}}      & \multicolumn{2}{c}{{\bf TSA+}}   \\ \cline{2-5} 
\multicolumn{1}{l|}{}                 & MlucI & \multicolumn{1}{c|}{HaeIII} & 18h  & \multicolumn{1}{c|}{24h}  \\ \hline
\multicolumn{1}{|l|}{Reads (Million)} & 10    & \multicolumn{1}{c|}{10.4}   & 42.2 & \multicolumn{1}{c|}{24.2} \\ \cline{2-5} 
\multicolumn{1}{|l|}{Duplicated (\%)} & 62.8  & \multicolumn{1}{c|}{74.0}   & 71.2 & \multicolumn{1}{c|}{84.4} \\ \hline
\end{tabular}
\end{table}

We found the proportion of reads mapped to the fair region chromosome, chromosome 5 in this case, fell below the prescribed level of $50\%$. Looking at three experiments (two from the untreated control), we find instead that between approximately $10$--$20\%$ of all reads mapped to the bait chromosome (Fig. \ref{fig:4cchromosomes}). While this is still a clear enrichment over non-bait chromosomes, it suggests the assay results suffer from either increased \emph{trans}-contact noise or decreased \emph{cis}-contact enrichment around the bait region. 

\begin{figure}
\begin{center} 
\includegraphics[width=5.4in]{figs/4c_chromosomes.pdf}
\captionsetup{width=\textwidth} 
\caption[ The bait chromosome is enriched for 4C sequencing reads. ]{ {\bf The bait chromosome is enriched for 4C sequencing reads. }
Chromosome 5 is visibly enriched for 4C reads as it  contains the ZRS bait region (or viewpoint). Untreated control samples (\emph{TSA-}) were assayed with two different secondary restriction enzymes (4-cutters HaeIII and MlucI).
}\label{fig:4cchromosomes}
\end{center} 
\end{figure} 

Lower than expected levels of both sequence duplication and bait chromosome enrichment suggest loss of signal around the bait region itself. This is the area where we'd expect both very high levels of duplication (identical restriction fragment pairings between nearby genomic locations) and a majority of all sequencing reads, driving the overall chromosome enrichment. The precise reason for the discrepancy is unclear but suggests the results may have a lower signal-to-noise ratio than has previously been achievable in 4C experiments.\cite{Stadhouders2013} Potentially the signal-to-noise ratio could be improved by utilising a double cross-linking procedure such as that used in \citet{Lin2012}

\subsection{3D modelling with 5C data}

All-vs-all contacts measured either genome-wide in the case of Hi-C, or over a defined region with 5C, can be used to infer the trajectory of chromatin fibres in three-dimensions through a variety of methods (e.g. \cite{Bau2011a, Hu2013a, Varoquaux2014a, Lesne2014, Trieu2014, Peng2013, Ay2014b, Caudai2015}). 5C data was generated over this same SHH--ZRS region (Fig. \ref{fig:shhtad}) with the aim of developing a multi-point perspective on local chromatin conformation beyond that available from 4C data.

We used this 5C experimental data in combination with a particular three-dimensional inference program  (\texttt{AutoChrom3D}\cite{Peng2013}) in an attempt to compare polymer trajectories in TSA treated and untreated 14fp mouse cells.

\begin{figure}
\begin{center} 
\includegraphics[width=5in]{figs/5c_3d.png}
\captionsetup{width=\textwidth} 
\caption[ Inferred 3D structure of the SHH--ZRS TAD. ]{ {\bf Inferred 3D structure of the SHH--ZRS TAD. }
Placheolder.
}\label{fig:5c3d}
\end{center} 
\end{figure} 

\section{5C in the HoxD region}

HoxD is another well-studied genetic system involved in limb development and under the control of known enhancers. In this experiment, our collaborators were interested in the chromatin conformations of HoxD13 loci in both the anterior and posterior developing limb bud, particularly how and where the two differed. To this end, our collaborators performed 5C for two biological replicates in anterior and posterior limb bud cell lines, and my contribution was to call differential contacts between the two conditions.

% Region gets a mention in Bickmore 2013 review 

% files for this under iain on ext HD
% relevant publications: http://dev.biologists.org/content/139/17/3157.full.pdf+html

%Iain's thesis: https://www.era.lib.ed.ac.uk/handle/1842/8056

\subsection{Differential contacts}

% raw diff: fold change?

\begin{figure}
\begin{center} 
\includegraphics[width=\textwidth]{figs/5cfc.pdf}
\captionsetup{width=\textwidth} 
\caption[Raw differences between anterior and posterior 5C interactions.]{ {\bf Raw differences between anterior and posterior 5C interactions. }
Placeholder
}\label{fig:5cfc}
\end{center} 
\end{figure} 

% statistical test of differential contacts

\begin{figure}
\begin{center} 
\includegraphics[width=\textwidth]{figs/5cdiff.pdf}
\captionsetup{width=\textwidth} 
\caption[Differential 5C contacts between 5C experiments over the HoxD locus in anterior and posterior limb bud cells.]{ {\bf Differential 5C contacts between 5C experiments over the HoxD locus in anterior and posterior limb bud cells. }
Scatterplots compare anterior and posterior interaction frequencies ($IF$, \emph{left}) and significantly different contacts are highlighted in the 5C heatmaps with colour-matched lines and squares (\emph{right}). No differential contacts were found over both repeats (Sets 1 and 2).
}\label{fig:5cdiff}
\end{center} 
\end{figure} 


\subsection{5C / Hi-C comparison}

% Precedent for comparing 5C and Hi-C : Phillips-Cremins2013

\begin{figure}
\begin{center} 
\includegraphics[width=\textwidth]{figs/5chic.pdf}
\captionsetup{width=\textwidth} 
\caption[Comparison of 5C and Hi-C over the HoxD region in different cell types.]{ {\bf Comparison of 5C and Hi-C over the HoxD region in different cell types. }
Previously-published Hi-C data\cite{Dixon2012} for two cell types is shown alongside novel 5C data in limb bud lines. Chances in the total contacts for a row or column are shown (\emph{lower}) with a fitted loess model. Significance outliers are labelled.
}\label{fig:5cdiff}
\end{center} 
\end{figure} 


\ifstandalone
\begin{small}
\bibliography{/Users/benmoore/Documents/library,/Users/benmoore/Documents/customrefs}
\end{small}
\fi

\end{document}
