\documentclass{beamer}
\usepackage{tikz}
\usepackage{wrapfig}
\usepackage{float}
\usepackage{multicol}
\usepackage{amsmath}
\usepackage[listings,theorems]{tcolorbox}
\setbeamercolor*{title}{use=structure,fg=white,bg=black}
\setbeamercolor*{author}{use=structure,fg=white}
\setbeamercolor*{date}{use=structure,fg=white}
\setbeamercolor{whitetext}{fg=white}

\title[Unravelling higher order chromatin organisation through statistical analysis] % (optional, only for long titles)
{ {\large Unravelling higher order chromatin organisation through statistical analysis } }
%\subtitle{ {\normalsize Second year report } }
\author[ \insertlogo{} ] 
{\vspace{15ex} \\
Ben Moore
\vspace{-4ex} }
% \institute[IGMM] 
% {
%   MRC HGU, IGMM
% }
\date{19$^{\mathrm{th}}$ November 2015}
\logo{
  \includegraphics[width=1.35cm]{figs/logo.png}
}
%\usetheme{Hannover}
\usetheme{Marburg}
\usecolortheme{dolphin}
%\usecolortheme{dove}

 %% End preamble

\begin{document}


{
\usebackgroundtemplate{\includegraphics[width=.85\paperwidth]{figs/hicbg_3.png}}
%\setbeamertemplate{sidebar left}{\logo{}}
\begin{frame}
%\hspace{-20em}
\titlepage
\end{frame}
}

% ----- Begin introduction

\section{Introduction}

\begin{frame}{Introduction}

Broad aim: investigate the relationship between structure and function of the genome

\vspace{2em}
Some specific questions:

\begin{enumerate}
\item How does higher order chromatin structure compare across human cell types?\item Can we predict higher order chromatin structure from locus-level features?\item How do the characteristics of boundaries demarcating higher order domains vary between cell types and domain classes?
\end{enumerate}

\end{frame}

\begin{frame}{What's known about genome structure}
\centering
\includegraphics[width=.8\textwidth]{../figs/genome_org.png}

\end{frame}

\begin{frame}{Hi-C: a genome-wide chromosome conformation assay }

\centering
\includegraphics[width=.8\textwidth]{../figs/hic.png}

\end{frame}


\begin{frame}{Chromosome compartments from Hi-C data }

\centering
\includegraphics[width=.6\textwidth]{../figs/eigcalc.png}

\end{frame}

\begin{frame}{ Strategy }

{\small

\begin{itemize}
\item Integrate publicly available Hi-C data
\item Uniformly reprocess each dataset 
\item Call compartments, TADs from reprocessed data
\item Integrate cell-matched ENCODE epigenomic data
\end{itemize}

Then:
\begin{enumerate}
\item Compare/contrast cell types after reprocessing
\item Attempt predictive models of compartments and TADs from epigenomic features
\item Analyse boundary composition in terms of epigenomic features
\end{enumerate}

Related collaborative work:
\begin{enumerate}
\setcounter{enumi}{3}
\item Investigate concept of "metaTADs"
\item Analyse conformation changes at specific locus of interest 
\end{enumerate}
}

\end{frame}

% ----- End of introduction

% ----- Begin Reanalysis of Hi-C

\section{Reanalysis of Hi-C datasets}

{
\usebackgroundtemplate{\includegraphics[width=.85\paperwidth]{figs/hicbg_3.png}}
\begin{frame}{}
\begin{tcolorbox}[colback=blue!40!black,colframe=blue!40!black]
\begin{center}
{\usebeamercolor[fg]{whitetext}
{\small Results 1: } \\
 Reanalysis of Hi-C data }
\end{center}
\end{tcolorbox}
\end{frame}
}

\begin{frame}{Reanalysis of Hi-C datasets}
Very different sequencing depths between the input datasets: \\

\vspace{2em}

\centering
\includegraphics[width=.9\textwidth]{../figs/hicnorm.png}

\end{frame}

\begin{frame}{Reanalysis of Hi-C datasets}
Despite this, reprocessed Hi-C data is well-correlated: \\

\vspace{2em}

{\centering
\includegraphics[width=.9\textwidth]{../figs/blowout.png}
}
\vspace{2em}

Justifies going forward with between cell-line analysis

\end{frame}

\begin{frame}{Improved compartment calling algorithm}

\begin{columns}

\column{.4\textwidth}
\centering
\includegraphics[width=.85\textwidth]{../figs/hmm_flow.pdf}

\column{.6\textwidth}
\centering
\includegraphics[width=\textwidth]{../figs/hmm_calls.png}

\end{columns}

\end{frame}

\begin{frame}{Regions of variable compartment structure}

Regions of open, active compartment in one cell type but closed, repressive in the other two are enriched for cell type specific enhancer chromatin states: \\

\vspace{2em}

\centering
\includegraphics[width=.55\textwidth]{../figs/enhancerstates.pdf}

\end{frame}

\begin{frame}{Regions of variable compartment structure}

Some harbour genes with known cell type specific function

\vspace{1em}

\centering
\includegraphics[width=.55\textwidth]{../figs/examplervs.png}

\end{frame}



% ----- End Reanalysis of Hi-C

\section{Integrative modelling}

{
\usebackgroundtemplate{\includegraphics[width=.85\paperwidth]{figs/hicbg_3.png}}
\begin{frame}{}
\begin{tcolorbox}[colback=blue!40!black,colframe=blue!40!black]
\begin{center}
{\usebeamercolor[fg]{whitetext}
{\small Results 2: } \\
 Integrative modelling as a tool \\ to explore biological systems}
\end{center}
\end{tcolorbox}
\end{frame}
}

% ----- Begin Integrative modelling

\begin{frame}{Integrative modelling}{p67}

Chromosome compartment profiles can be accurately predicted and models generalise across cell types: \\

\vspace{2em}

\centering
\includegraphics[width=.65\textwidth]{figs/f2.png}
\includegraphics[width=.35\textwidth]{figs/f3.png}

\end{frame}

\begin{frame}{Integrative modelling}

Model dissection a focus of much of this chapter: \\

\vspace{2em}

\begin{itemize}
\item Variable importance
\begin{itemize}
\item Varies between cell type models but multicollinearity in feature set
\item In some cases interpretable (p70), many of the usual suspects of chromatin conformation
\end{itemize}
\item Comparing machine learning methods
\item Regularised models
\end{itemize}

\vspace{1em}

\end{frame}

\begin{frame}{Integrative modelling}

Most results in these two chapters were published:

\vspace{2em}
\centering
\includegraphics[width=.9\textwidth]{../figs/paper_screenshot.png}
\vspace{2em}

\end{frame}

\section{Boundaries}

{
\usebackgroundtemplate{\includegraphics[width=.85\paperwidth]{figs/hicbg_3.png}}
\begin{frame}{}
\begin{tcolorbox}[colback=blue!40!black,colframe=blue!40!black]
\begin{center}
{\usebeamercolor[fg]{whitetext}
{\small Results 3: } \\
 Chromatin domain boundaries}
\end{center}
\end{tcolorbox}
\end{frame}
}

% ----- Begin boundaries

\begin{frame}{Boundaries}

TAD boundaries have been shown to be enriched for various epigenomic marks. \\

\vspace{2em}

\begin{itemize}
\item Which of these boundary enrichments are statistically significant relative to a null model?
\item Are the same features enriched at chromatin compartment boundaries?
\item Are these enrichments shared across cell types?
\end{itemize}

\end{frame}

\begin{frame}{Boundary enrichment analysis}

\centering

\includegraphics[width=.8\textwidth]{../figs/boundarysummary.pdf}

\end{frame}

\begin{frame}{Predicting TAD boundaries}{p96}

Given these enrichments exist, can we predict TAD boundaries in lieu of Hi-C data using epigenomic features? \\

\vspace{2em}

\centering
\includegraphics[width=.5\textwidth]{../figs/tadpred_auroc.pdf}


\end{frame}

\begin{frame}{Predicting TAD boundaries}

In this case, more consistent and clear results from variable importance:

\vspace{2em}

\centering
\includegraphics[width=\textwidth]{../figs/tadpred_varimp.pdf}

\end{frame}


\begin{frame}{MetaTADs}

Collaborator's finding: \\
\begin{quote}
Neighbour joining of interacting TADs appears to 
capture a novel layer of chromatin organisation: metaTADs. This attractive
concept that offers a framework for fractal 3D structure and links TADs with compartments.
 \end{quote}

\vspace{2em}

My contributions: \\
\begin{itemize}
\item Analysis of MetaTAD boundary enrichments as per TADs and compartments
\item Analysis of the co-incidence of MetaTAD boundaries and LAD boundaries
\end{itemize}

\vspace{2em}

\end{frame}

\begin{frame}{MetaTAD boundary enrichments}{p100}

\centering
\includegraphics[width=.75\textwidth]{../figs/metatad_cutoffenrich.pdf}

\end{frame}


\section{Collaborations}
\begin{frame}{Side project}
Collaborating with Adam Douglas (Bob Hill's group) in analysing his
3C-seq (4C) data. Their hypothesis: \\

\vspace{1em}

\centering

\includegraphics[width=.8\textwidth]{figs/tsa.png}
\end{frame}

\begin{frame}{Side project}
Initial results: \\

\vspace{1em}

\centering

\includegraphics[width=.9\textwidth]{figs/shh.pdf} \\
\end{frame}

\begin{frame}{Side project}
Zoomed: \\

\vspace{1em}

\centering

\includegraphics[width=.9\textwidth]{figs/zoomSHH.pdf} \\

Incoming: repeats, 5C / Capture-C (?), FAIRE-seq \dots
\end{frame}

\begin{frame}{Side project}
All looks good except\dots \\

\vspace{1em}

\centering

\includegraphics[width=.9\textwidth]{figs/csomeprop.pdf} \\

\end{frame}

\section{Next project}
\begin{frame}{Next project}
  So far haven't looked at any expression data, but CAGE available for
  each cell line. \\

\vspace{1em}

Initial idea:\\

\vspace{1em}

\begin{itemize}
\item Investigate TADs as ``regulons''---new paper reports $~20\%$
  domains act as ``discrete regulatory units'' with relatively
  homogenous epigenetic states.\footnote{ {\tiny Le Dily \emph{et al.} (2014) Distinct structural transitions of chromatin correlate with
  coordinated hormone-induced gene regulation. \emph{Genes and
    Development}, {\bf 28}:2151-62.} } 
\end{itemize}

\vspace{1em}

Can I find evidence for this with my data?
\end{frame}

\begin{frame}{Next project}
\centering
\includegraphics[width=.3\textwidth]{figs/venn.pdf} \\
\includegraphics[width=.5\textwidth]{figs/hgd.pdf}
\includegraphics[width=.5\textwidth]{figs/hist.pdf} 
\end{frame}

\begin{frame}{Next project}
\begin{itemize}
\item Higher resolution Hi-C now available, better for identifying domains
\item Improved methods of calling domains, where / why are some
  well-conserved between cell types?
\item Mouse data also available
\end{itemize}
\end{frame}


% \section{Year summary}
% \begin{frame}{Year summary}
% \vspace{-1em}
% \begin{enumerate}
% \item Completed main project on integrating uniformly-processed Hi-C and
%   ENCODE datasets (\emph{in review})
% \includegraphics[width=.8\textwidth]{figs/inreview.png}
% \item Presented related posters at ISMB (Boston) and Genome
%   Informatics (Cambridge); spoke at Edinburgh Bioinformatics
% \item Ongoing collaborations with wet-lab researchers (particularly
%   Adam Douglas in Bob Hill's group)
% \end{enumerate}
% \end{frame}

\section{Thesis plan}
\begin{frame}{Thesis plan p1}
\begin{enumerate}
\item Introduction
\item Methods
\item {\bf Modelling transcription and chromatin} Replicating and
  extending ENCODE project predicting txn output; adapt techniques to
  genome organisation
\item {\bf Model dissection} How the chromatin structure models
  cross-apply; regularised models; variable importance
\item {\bf Boundaries} Comparing TADs/compartments across boundaries;
  ``super bounds''; Giemsa bands
\item \dots
\end{enumerate}

\end{frame}

\begin{frame}{Thesis plan p2}
\begin{enumerate}
\setcounter{enumi}{5}
\item {\bf Investigating the function of self-interacting domains}
  Work from future project (very early stages); possibly two chapters
  from the next year
\item {\bf C-methods collaborations} Write-up Hill lab collaborations,
  possibly include other minor collabs
\item Discussion
\item End materials, code
\end{enumerate}

\end{frame}


\section{Acknowledgements}

{
\usebackgroundtemplate{\includegraphics[width=.85\paperwidth]{figs/hicbg_3.png}}
\begin{frame}{}
\begin{tcolorbox}[colback=blue!40!black,colframe=blue!40!black]
\begin{center}
{\usebeamercolor[fg]{whitetext}
{\small Thanks to supervisors:} \\ Colin Semple and
Stuart Aitken
}
\end{center}
\end{tcolorbox}

\end{frame}
}


\end{document}