
<h1 id="input-data">Input data</h1>
<p>Some more stuff here.</p>
<h2 id="hi-c-data">Hi-C data</h2>
<p>Raw Hi-C reads were downloaded from three published datasets through GEO or the SRA with identifiers: GSE35156 (H1 hESC), GSE18199 (K562) and SRX030113 (GM12878). These paired reads were mapped independently to theand mapped to the genome (hg19/GRCh37). Mapping was performed using the <code>hiclib</code> software package and <code>bowtie2</code> with the <code>--very-sensitive</code> flag. Mapped reads were then binned into contact maps and iteratively corrected. The <code>hiclib</code> software was also used for eigenvector expansion of each intrachromosomal contact map, performed independently on each chromosome arm.</p>
<h2 id="locus-level-features">Locus-level features</h2>
<p>Genome-wide ChIP-seq datasets for: 22 DNA binding proteins and 10 histone marks were made available by the ENCODE consortium along with DNase I hypersensitivity and H2A.z occupancy, for each of the Tier 1 ENCODE cell lines used in this work: H1 hESC, K562 and GM12878. These data were pre-processed using MACSv2 to produce fold-change relative to input chromatin. GC content was also calculated and used in the featureset.</p>
<h1 id="modelling">Modelling</h1>
<h2 id="sec:rf">Random Forest</h2>
<p>Random Forest regression was used as implemented in the R package <code>randomForest</code>. Parameters of <span class="math">$mtry = \frac{n}{3} = 12$</span> and <span class="math"><em>n</em><em>t</em><em>r</em><em>e</em><em>e</em><em>s</em> = 200</span> were assumed as they approximate the defaults and are known to be largely insensitive.</p>
<p>Variable importance within Random Forest regression models was measured using mean decrease in accuracy in the out-of-bag (OOB) sample. This represents the average difference (over the forest) between the accuracy of a tree with permuted and unpermuted versions of a given variable, in units of mean squared error (MSE).</p>
<h2 id="model-performance">Model performance</h2>
<p>The effectiveness of the modelling approach was measured by four different metrics. Prediction accuracy was assessed by the Pearson correlation coefficient between the predicted and observed eigenvectors (determined by 10-fold cross-validation), and the root mean-squared error (RMSE) of the same data. Classification error, when predictions where thresholded into <span class="math"><em>A</em> ≥ 0; <em>B</em> &lt; 0</span>, was also calculated using accuracy (% correct classifications or True Positives) and area under the receiver operating characteristic (AUROC) curve. Together these give a comprehensive overview of the model performance, both in terms of regression accuracy of the continuous eigenvector, and in how that same model could be used to label discrete chromatin compartments.</p>
<p>For cross-application of cell type specific models, a single Random Forest regression model was learned from all 1 Mb bins for a given cell type. This was then used to predict all bins from each of the other two cell types.</p>
<h1 id="variable-regions">Variable regions</h1>
<h2 id="stratification-by-variability">Stratification by variability</h2>
<p>Median absolute deviation (MAD) was chosen as a robust measure of the variability in a given 1 Mb block between the three primary cell types used in this work: H1, K562 and GM12878. Blocks were ranked by this measure and split into thirds that represented “low” variability (the third of blocks with the lowest MAD), “mid” and “high” variability. Each subgroup was then independently modelled using the previously-described Random Forest approach.</p>
<p>“Flipped” regions are those whose compartment state differs in one cell type relative to the other two. For example, if a 1 Mb bin was classified as “open” in H1 hESC and ``closed'' in both K562 and GM12878, this is said to be a “flipped” compartment (to open).</p>
<h2 id="enhancer-enrichment">Enhancer enrichment</h2>
<p>Enhancer annotations were collected from the ChromHMM / SegWay combined annotations in each cell type. Enhancers were considered “shared” if there was an overlapping enhancer annotation in either of the two other cell types, and labelled as “tissue-specific” otherwise.</p>
<h1 id="boundaries">Boundaries</h1>
<h2 id="tads">TADs</h2>
<p>TAD boundaries were called using the software provided in  using their recommended parameters. For the generation of boundary profiles, the same parameters were used: input features were averaged into 40 kb bins spanning <span class="math"> ± 500</span> kb from the boundary centre.</p>
<h2 id="sec:compartments">Compartments</h2>
<p>Compartment boundaries were called by first training a two-state hidden Markov model (HMM) on the compartment eigenvector and then using the Viterbi algorithm to predict the most likely state sequence that produced the observed values. The point at which transitions occurred between states was taken as a boundary which was then extended <span class="math"> ± 1.5</span> Mb to give a 3 Mb window in which a boundary was though to occur.</p>
<p>To test for the enrichment or depletion of a chromatin feature over a given boundary, a two tailed Mann-Whitney test was used to compare the boundary bin with the ten outermost bins of the window (5 from either side). The significance level at ￼<span class="math"><em>α</em> = 0.01</span> was then Bonferonni-adjusted for multiple testing correction, and results with <em>p</em>-values exceeding this threshold were deemed significantly enriched or depleted at a given boundary.</p>
<h1 id="giemsa-band-comparison">Giemsa band comparison</h1>
<p>Cytogenic band data and Giemsa stain results were downloaded from the UCSC genome browser (table <code>cytoBandIdeo</code>). The genomic co-ordinates are an approximation of cytogenic band data inferred from a large number of FISH experiments.</p>
<p>To compare G-band boundaries with our compartment data, we allowed for a <span class="math"> ± 500</span> kb inaccuracy in G-band boundary. For each G-band boundary, the minimum absolute distance to any compartment or TAD boundary was calculated for each cell type. To generate a null model, </p>
